Description: fix a spelling mistake
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-04-23

--- a/lib/HTML/Tree/AboutObjects.pod
+++ b/lib/HTML/Tree/AboutObjects.pod
@@ -279,7 +279,7 @@
 read or set the attribute's value.  Some of the object's attributes are
 private, meaning you'll never see them documented because they're not
 for you to read or write; but most of the object's documented attributes
-are at least readable, and usually writeable, by you.  Net::FTP objects
+are at least readable, and usually writable, by you.  Net::FTP objects
 are a bit thin on attributes, so we'll use objects from the class
 Business::US_Amort for this example.  Business::US_Amort is a very
 simple class (available from CPAN) that I wrote for making calculations
