Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: spelling mistakes
Forwarded: no
Last-Update: 2011-05-03
--- a/IP.pm
+++ b/IP.pm
@@ -1201,7 +1201,7 @@
 # Purpose           : Split a prefix into IP and prefix length
 # Comments          : If it was passed a simple IP, it just returns it
 # Params            : Prefix
-# Returns           : IP, optionnaly length of prefix
+# Returns           : IP, optionally length of prefix
 sub ip_splitprefix {
     my $prefix = shift;
 
@@ -2012,7 +2012,7 @@
 
 Return the current object error string. The error string is set whenever one 
 of the methods produces an error. Also, a global, class-wide B<Error()> 
-function is avaliable.
+function is available.
 
 C<warn ($ip-E<gt>error());>
 
@@ -2335,7 +2335,7 @@
 If it was passed a simple IP, it just returns it.
 
     Params  : Prefix
-    Returns : IP, optionnaly length of prefix
+    Returns : IP, optionally length of prefix
 
 C<($ip,$len) = ip_splitprefix ($prefix)>
 
@@ -2504,7 +2504,7 @@
 
 The Math::BigInt library is needed for functions that use integers. These are
 ip_inttobin, ip_bintoint, and the size method. In a next version, 
-Math::BigInt will become optionnal.
+Math::BigInt will become optional.
 
 =head1 AUTHORS
 
