Description: Use -fPIC, minor compile fixes.
Debian-Bugs: #258991, #259440, #258680
Author: Matthias Urlichs <smurf@debian.org>

--- a/RTjpeg/Makefile.PL
+++ b/RTjpeg/Makefile.PL
@@ -3,6 +3,7 @@ use ExtUtils::MakeMaker;
 WriteMakefile(
     NAME	 => 'Video::RTjpeg',
     VERSION_FROM => 'RTjpeg.pm',
+	CCFLAGS  => '-fPIC',
     OBJECT       => '$(BASEEXT)$(OBJ_EXT) codec/RTjpeg.o',
 );
 
@@ -11,6 +12,6 @@ codec/RTjpeg.c: codec/compose.sh
 	cd codec && . ./compose.sh
 
 codec/RTjpeg.o: codec/RTjpeg.c codec/RTjpeg.h
-	\$(CCCMD) \$(CCCDLFLAGS) -o \$@ \$<
+	\$(CCCMD) \$(CCCDLFLAGS) -fPIC -o \$@ \$<
 EOF
 }
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -9,6 +9,7 @@ WriteMakefile(
     'NAME'	=> 'Video::Frequencies',
     DISTNAME	=> 'Video-Capture-V4l',
     'VERSION_FROM' => 'V4l/V4l.pm',
+	CCFLAGS  => '-fPIC',
     linkext	=> {LINKTYPE => ''},
     DIR		=> ['V4l','VBI','RTjpeg'],
 );
--- a/VBI/VBI.xs
+++ b/VBI/VBI.xs
@@ -482,7 +482,7 @@ decoder_decode (decoder *dec, UI y, u8 *
                     case 0x55:
                       break;
                     default:
-                      ; /* no teletext page */
+                      break; /* no teletext page */
                     }
 
                   if (data[1] == 0x55)
