--- a/RTjpeg/codec/modules/RTb2s_raw.c
+++ b/RTjpeg/codec/modules/RTb2s_raw.c
@@ -2,7 +2,7 @@
 {
  int ci=1, co=1, tmp;
 
- (u8)strm[0]=(u8)(data[RTjpeg_ZZ[0]]>254) ? 254:((data[RTjpeg_ZZ[0]]<0)?0:data[RTjpeg_ZZ[0]]);
+ strm[0]=(u8)(data[RTjpeg_ZZ[0]]>254) ? 254:((data[RTjpeg_ZZ[0]]<0)?0:data[RTjpeg_ZZ[0]]);
  
  for(ci=1; ci<=63; ci++)
   if(data[RTjpeg_ZZ[ci]]>0)
--- a/VBI/VBI.xs
+++ b/VBI/VBI.xs
@@ -482,7 +482,7 @@
                     case 0x55:
                       break;
                     default:
-                      ; /* no teletext page */
+                      break; /* no teletext page */
                     }
 
                   if (data[1] == 0x55)
