Author: Andreas Jochens <aj@andaco.de>
Bug-Debian: #318547
Description: fix FTBFS ("error: invalid lvalue in assignment")

--- a/RTjpeg/codec/modules/RTb2s_raw.c
+++ b/RTjpeg/codec/modules/RTb2s_raw.c
@@ -2,7 +2,7 @@ int RTjpeg_b2s(s16 *data, s8 *strm, u8 b
 {
  int ci=1, co=1, tmp;
 
- (u8)strm[0]=(u8)(data[RTjpeg_ZZ[0]]>254) ? 254:((data[RTjpeg_ZZ[0]]<0)?0:data[RTjpeg_ZZ[0]]);
+ strm[0]=(u8)(data[RTjpeg_ZZ[0]]>254) ? 254:((data[RTjpeg_ZZ[0]]<0)?0:data[RTjpeg_ZZ[0]]);
  
  for(ci=1; ci<=63; ci++)
   if(data[RTjpeg_ZZ[ci]]>0)
