From: Ansgar Burchardt <ansgar@43-1.org>
Date: Sat, 05 Jun 2010 12:27:02 +0900
Subject: Fix spelling error
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=58154

--- libnet-sieve-perl.orig/lib/Net/Sieve.pm
+++ libnet-sieve-perl/lib/Net/Sieve.pm
@@ -883,7 +883,7 @@
 
 =head1 BUGS
 
-I don't try plain text or client certificate authentification. 
+I don't try plain text or client certificate authentication.
 
 You can debug TLS connexion with openssl :
    openssl s_client -connect your.server.org:2000 -tls1 -CApath /etc/apache/ssl.crt/somecrt.crt -starttls smtp
