Subject: to fix the path of the rc file in the manpage
Author: Jonathan Yu <jawnsey@cpan.org>
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
Last-Update: 2010-08-15
--- a/bin/prove
+++ b/bin/prove
@@ -73,7 +73,7 @@
 
 =head1 NOTES
 
-=head2 .proverc
+=head2 F<~/.proverc> / F<./.proverc>
 
 If F<~/.proverc> or F<./.proverc> exist they will be read and any
 options they contain processed before the command line options. Options
