Description: spelling mistake
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-12-26

--- a/lib/POEx/Role/SessionInstantiation/Meta/Session/Magic.pm
+++ b/lib/POEx/Role/SessionInstantiation/Meta/Session/Magic.pm
@@ -177,7 +177,7 @@
 
 =head2 overload "", !=, ==
 
-Stringification, and numeric comparison are overriden so that we can fool POE
+Stringification, and numeric comparison are overridden so that we can fool POE
 into thinking that our inject reference is actually the same as the old 
 reference.
 
