Author: David Paleino <d.paleino@gmail.com>
Description: No description.
--- a/lib/SVG/Element.pm
+++ b/lib/SVG/Element.pm
@@ -745,7 +745,7 @@
 
     # populate the script tag with cdata
     # be careful to manage the javascript line ends.
-    # qq|text| or qq§text§ where text is the script 
+    # qq|text| or qq\[sc]text\[sc] where text is the script
     # works well for this.
 
     $tag->CDATA(qq|function d(){
@@ -772,7 +772,7 @@
 
     # populate the style tag with cdata
     # be careful to manage the line ends.
-    # qq|text| or qq§text§ where text is the script
+    # qq|text| or qq\[sc]text\[sc] where text is the script
     # works well for this.
 
     $tag1->CDATA(qq|
@@ -1218,7 +1218,7 @@
 
 B<Example:>
 
-      my $text = qq§
+      my $text = qq\[sc]
         var SVGDoc;
         var groups = new Array();
         var last_group;
@@ -1236,7 +1236,7 @@
             append_group(1, 4, 6); // group 0
             append_group(5, 4, 3); // group 1
             append_group(2, 3);    // group 2
-        }§;
+        }\[sc];
         $SVG->script()->CDATA($text);
 
 
--- a/lib/SVG/Manual.pm
+++ b/lib/SVG/Manual.pm
@@ -218,7 +218,7 @@
 =head1 CREDITS
 
 Peter Wainwright, Excellent ideas, beta-testing, writing SVG::Parser and much of SVG::DOM.  Fredo, http://www.penguin.at0.net/~fredo/ - provided example code and initial feedback for early SVG.pm versions and the idea of a simplified svg generator.
-Adam Schneider, Brial Pilpré, Ian Hickson
+Adam Schneider, Brial Pilpr\['e], Ian Hickson
 Martin Owens - SVG::DOM improvements in version 3.34
 
 =head1 EXAMPLES
@@ -607,7 +607,7 @@
 
     # populate the script tag with cdata
     # be careful to manage the javascript line ends.
-    # qq|text| or qq§text§ where text is the script 
+    # qq|text| or qq\[sc]text\[sc] where text is the script
     # works well for this.
     #make sure to use the CAPITAL CDATA to poulate the script.
     $tag->CDATA(qq|function d(){
@@ -843,7 +843,7 @@
 
 B<Example:>
 
-      my $text = qq§
+      my $text = qq\[sc]
         var SVGDoc;
         var groups = new Array();
         var last_group;
@@ -861,7 +861,7 @@
             append_group(1, 4, 6); // group 0
             append_group(5, 4, 3); // group 1
             append_group(2, 3);    // group 2
-        }§;
+        }\[sc];
         $svg->script()->CDATA($text);
 
 
--- a/lib/SVG/Extension.pm
+++ b/lib/SVG/Extension.pm
@@ -4,6 +4,12 @@
 use vars qw(@ISA $VERSION @TYPES %TYPES);
 $VERSION = "2.50";
 
+=head1 NAME
+
+SVG::Extension - additional methods
+
+=cut
+
 # although DTD declarations are not elements, we use the same API so we can
 # manipulate the internal DTD subset using the same methods available for
 # elements. At this state, all extensions are the same object class, but
