Description: don't compile with -lssl
Author: Damyan Ivanov <dmn@debian.org>
Origin: vendor
Forwarded: not-needed
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -11,7 +11,7 @@
     'DISTNAME' => 'Crypt-OpenSSL-RSA',
     'VERSION_FROM' => 'RSA.pm', # finds $VERSION
     'OBJECT' => 'RSA.o',
-    'LIBS'	=> ['-lssl -lcrypto'],   # e.g., '-lm'
+    'LIBS'	=> ['-lcrypto'],   # e.g., '-lm'
     'PREREQ_PM' => { 'Crypt::OpenSSL::Random' => 0 },
     'DEFINE'	=> '-DPERL5 -DOPENSSL_NO_KRB5',
     # perl-5.8/gcc-3.2 needs -DPERL5, and redhat9 likes -DOPENSSL_NO_KRB5
