Description: fix POD spelling
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: no
--- a/lib/Math/Symbolic/MiscCalculus.pm
+++ b/lib/Math/Symbolic/MiscCalculus.pm
@@ -98,7 +98,7 @@
 This function (symbolically) computes the nth-degree Taylor Polynomial
 of a given function. Generally speaking, the Taylor Polynomial is an
 n-th degree polynomial that approximates the original function. It does
-so particularily well in the proximity of a certain point x0.
+so particularly well in the proximity of a certain point x0.
 (Since my mathematical English jargon is lacking, I strongly suggest you
 read up on what this is in a book.)
 
