Author: Carl Fürstenberg <azatoth@gmail.com>
Description: Add missing whatis entries.
Index: libgtk2-sexy-perl/xs/SexyIconEntry.xs
===================================================================
--- libgtk2-sexy-perl.orig/xs/SexyIconEntry.xs	2008-07-12 20:31:01.000000000 +0200
+++ libgtk2-sexy-perl/xs/SexyIconEntry.xs	2008-07-12 20:34:16.000000000 +0200
@@ -4,6 +4,9 @@
 
 PROTOTYPES: disable
 
+=for object Gtk2::Sexy::IconEntry A GtkEntry with support for inline icons
+=cut
+
 GtkWidget *
 sexy_icon_entry_new (class)
 	C_ARGS:
Index: libgtk2-sexy-perl/xs/SexySpellEntry.xs
===================================================================
--- libgtk2-sexy-perl.orig/xs/SexySpellEntry.xs	2008-07-12 20:31:01.000000000 +0200
+++ libgtk2-sexy-perl/xs/SexySpellEntry.xs	2008-07-12 20:34:16.000000000 +0200
@@ -4,6 +4,10 @@
 
 PROTOTYPES: disable
 
+=for object Gtk2::Sexy::SpellEntry A GtkEntry with inline spell checking
+
+=cut
+
 GtkWidget *
 sexy_spell_entry_new (class)
 	C_ARGS:
Index: libgtk2-sexy-perl/xs/SexyUrlLabel.xs
===================================================================
--- libgtk2-sexy-perl.orig/xs/SexyUrlLabel.xs	2008-07-12 20:31:01.000000000 +0200
+++ libgtk2-sexy-perl/xs/SexyUrlLabel.xs	2008-07-12 20:34:16.000000000 +0200
@@ -4,6 +4,9 @@
 
 PROTOTYPES: disable
 
+=for object Gtk2::Sexy::UrlLabel A GtkLabel with support for embedded hyperlinks
+=cut
+
 GtkWidget *
 sexy_url_label_new (class);
 	C_ARGS:
Index: libgtk2-sexy-perl/xs/SexyTooltip.xs
===================================================================
--- libgtk2-sexy-perl.orig/xs/SexyTooltip.xs	2008-07-12 20:35:05.000000000 +0200
+++ libgtk2-sexy-perl/xs/SexyTooltip.xs	2008-07-12 20:48:57.000000000 +0200
@@ -4,6 +4,10 @@
 
 PROTOTYPES: disable
 
+=for object Gtk2::Sexy::ToolTip A flexible tooltip widget
+
+=cut
+
 GtkWidget *
 sexy_tooltip_new (class);
 	C_ARGS:
Index: libgtk2-sexy-perl/xs/SexyTreeView.xs
===================================================================
--- libgtk2-sexy-perl.orig/xs/SexyTreeView.xs	2008-07-12 20:47:39.000000000 +0200
+++ libgtk2-sexy-perl/xs/SexyTreeView.xs	2008-07-12 20:48:30.000000000 +0200
@@ -4,6 +4,9 @@
 
 PROTOTYPES: disable
 
+=for object Gtk2::Sexy::Treeview A treeview subclass which can display tooltips for items in the tree
+=cut
+
 GtkWidget *
 sexy_tree_view_new (class);
 	C_ARGS:
