From: Ansgar Burchardt <ansgar@43-1.org>
Date: Mon, 07 Jun 2010 21:03:02 +0900
Subject: Fix spelling errors
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=58195

--- libpath-class-perl.orig/lib/Path/Class/Dir.pm
+++ libpath-class-perl/lib/Path/Class/Dir.pm
@@ -638,7 +638,7 @@
 
 Returns the class which should be used to create file objects.
 
-Generally overriden whenever this class is subclassed.
+Generally overridden whenever this class is subclassed.
 
 =back
 
--- libpath-class-perl.orig/lib/Path/Class/File.pm
+++ libpath-class-perl/lib/Path/Class/File.pm
@@ -299,7 +299,7 @@
 
   my @lines = $file->slurp(chomp => 1);
 
-You may also use the C<iomode> paramter to pass in an IO mode to use
+You may also use the C<iomode> parameter to pass in an IO mode to use
 when opening the file, usually IO layers (though anything accepted by
 the MODE argument of C<open()> is accepted here).  Just make sure it's
 a I<reading> mode.
@@ -334,7 +334,7 @@
 
 Returns the class which should be used to create directory objects.
 
-Generally overriden whenever this class is subclassed.
+Generally overridden whenever this class is subclassed.
 
 =back
 
