Author: Angel Abad <angelabad@gmail.com>
Subject: Fix spelling error in manpage
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=54883 

--- a/lib/Sysadm/Install.pm
+++ b/lib/Sysadm/Install.pm
@@ -1478,7 +1478,7 @@
 =item C<pipe_copy($in, $out, [$bufsize])>
 
 Reads from $in and writes to $out, using sysread and syswrite. The
-buffer size used defaults to 4096, but can be set explicitely.
+buffer size used defaults to 4096, but can be set explicitly.
 
 =cut
 
