Description: fix a spelling mistake
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-05-24

--- a/IP.pm
+++ b/IP.pm
@@ -60,7 +60,7 @@
   );
 
   NOTE: NetAddr::IP::Util has a full complement of network address
-	utilites to convert back and from from binary to text.
+	utilities to convert back and from from binary to text.
 
 	inet_aton, inet_ntoa, ipv6_aton, ipv6_n2x, ipv6_n2d
 	inet_any2d, inet_n2dx, inet_n2ad, inetanyto6, ipv6to4
