Description: fix POD spelling errors
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=71131
Forwarded: yes
Author: Jonathan Yu <jawnsy@cpan.org>, intrigeri <intrigeri+debian@boum.org>
Reviewed-by: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2011-09-21

--- a/lib/Regexp/Common.pm
+++ b/lib/Regexp/Common.pm
@@ -678,7 +678,7 @@
 be made available - the wanted sets should be given as arguments to
 C<use>. Alternatively, it is also possible to indicate which sets of
 patterns should not be made available - those sets will be given as
-argument to the C<use> statement, but are preceeded with an exclaimation
+argument to the C<use> statement, but are preceded with an exclaimation
 mark. The argument I<no_defaults> indicates none of the default patterns
 should be made available. This is useful for instance if all you want
 is the C<pattern()> subroutine.
@@ -696,7 +696,7 @@
  use Regexp::Common qw /my_patterns/;
 
 Note that the default patterns will still be made available - only if
-you use I<no_defaults>, or mention one of the default sets explicitely,
+you use I<no_defaults>, or mention one of the default sets explicitly,
 the non mentioned defaults aren't made available.
 
 =head2 List of available patterns
@@ -786,7 +786,7 @@
 =item C<Can't create unknown regex: $RE{...}>
 
 Regexp::Common doesn't have a generator for the requested pattern.
-Often indicates a mispelt or missing parameter.
+Often indicates a misspelt or missing parameter.
 
 =item
 C<Perl %f does not support the pattern $RE{...}.
--- a/lib/Regexp/Common/comment.pm
+++ b/lib/Regexp/Common/comment.pm
@@ -669,9 +669,9 @@
 =item INTERCAL
 
 Comments in INTERCAL are single line comments. They start with one of
-the keywords C<NOT> or C<N'T>, and can optionally be preceeded by the
+the keywords C<NOT> or C<N'T>, and can optionally be preceded by the
 keywords C<DO> and C<PLEASE>. If both keywords are used, C<PLEASE>
-preceeds C<DO>. Keywords are separated by whitespace.
+precedes C<DO>. Keywords are separated by whitespace.
 
 =item J
 
@@ -733,9 +733,9 @@
 
 In C<M> (aka C<MUMPS>), comments start with a semi-colon, and last
 till the end of a line. The language specification requires the 
-semi-colon to be preceeded by one or more I<linestart character>s.
+semi-colon to be preceded by one or more I<linestart character>s.
 Those characters default to a space, but that's configurable. This
-requirement, of preceeding the comment with linestart characters is
+requirement, of preceding the comment with linestart characters is
 B<not> tested for. See
 L<ftp://ftp.intersys.com/pub/openm/ism/ism64docs.zip>,
 L<http://mtechnology.intersys.com/mproducts/openm/index.html>, and
