Description: spelling fix
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-06-12

--- a/lib/MIME/Parser.pm
+++ b/lib/MIME/Parser.pm
@@ -1821,7 +1821,7 @@
 
 B<Native I/O is much faster than object-oriented I/O.>
 It's much faster to use E<lt>$fooE<gt> than $foo-E<gt>getline.
-For backwards compatibilty, this module must continue to use
+For backwards compatibility, this module must continue to use
 object-oriented I/O in most places, but if you use L<parse()|/parse>
 with a "real" filehandle (string, globref, or subclass of IO::File)
 then MIME::Parser is able to perform some crucial optimizations.
--- a/lib/MIME/Parser/Filer.pm
+++ b/lib/MIME/Parser/Filer.pm
@@ -34,7 +34,7 @@
 
 =head2 Semi-public interface
 
-These methods might be overriden or ignored in some subclasses,
+These methods might be overridden or ignored in some subclasses,
 so they don't all make sense in all circumstances:
 
     ### Tweak the mapping from content-type to extension:
