http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=541165

--- a/lib/Module/Depends.pm
+++ b/lib/Module/Depends.pm
@@ -3,7 +3,7 @@
 use YAML::XS qw/LoadFile/;
 use Cwd qw( getcwd );
 use base qw( Class::Accessor::Chained );
-__PACKAGE__->mk_accessors(qw( dist_dir debug libs requires build_requires error ));
+__PACKAGE__->mk_accessors(qw( dist_dir debug libs requires build_requires test_requires configure_requires error ));
 our $VERSION = '0.14';
 
 =head1 NAME
@@ -41,6 +41,8 @@
         libs           => [],
         requires       => {},
         build_requires => {},
+        test_requires  => {},
+        configure_requires => {},
         error          => '',
     });
 }
@@ -51,7 +53,7 @@
 
 =head2 find_modules
 
-scan the C<dist_dir> to populate C<libs>, C<requires>, and C<build_requires>
+scan the C<dist_dir> to populate C<libs>, C<requires>, C<configure_requires>, C<test_requires>, and C<build_requires>
 
 =cut
 
@@ -78,6 +80,8 @@
         my $meta = LoadFile( $file );
         $self->requires( $meta->{requires} );
         $self->build_requires( $meta->{build_requires} );
+        $self->configure_requires( $meta->{configure_requires} );
+        $self->test_requires( $meta->{test_requires} );
         return $self;
     }
   }
--- a/lib/Module/Depends/Intrusive.pm
+++ b/lib/Module/Depends/Intrusive.pm
@@ -32,6 +32,8 @@
         my %args =  @_;
         $self->requires( $args{requires} || {} );
         $self->build_requires( $args{build_requires} || {} );
+        $self->test_requires( $args{test_requires} || {} );
+        $self->configure_requires( $args{configure_requires} || {} );
         bless {}, "Module::Depends::Intrusive::Fake::Module::Build";
     };
     local *Module::Build::subclass = sub { 'Module::Build' };
@@ -40,6 +42,7 @@
     my $WriteMakefile = sub {
         my %args = @_;
         $self->requires( $args{PREREQ_PM} || {} );
+        $self->build_requires ( $args{BUILD_REQUIRES} || {} );
         1;
     };
     local *main::WriteMakefile;
@@ -61,7 +64,7 @@
       all_from auto_install AUTOLOAD build_requires check_nmake include
       include_deps installdirs Makefile makemaker_args Meta name no_index
       requires WriteAll clean_files can_cc sign cc_inc_paths cc_files
-      cc_optimize_flags author license
+      cc_optimize_flags author license configure_requires test_requires
 
     );
     local *inc::Module::Install::AUTOLOAD = sub { 1 };
@@ -74,6 +77,14 @@
         my %deps = (@_ == 1 ? ( $_[0] => 0 ) : @_);
         $self->build_requires->{ $_ } = $deps{ $_ } for keys %deps;
     };
+    local *inc::Module::Install::test_requires = sub {
+        my %deps = (@_ == 1 ? ( $_[0] => 0 ) : @_);
+        $self->test_requires->{ $_ } = $deps{ $_ } for keys %deps;
+    };
+    local *inc::Module::Install::configure_requires = sub {
+        my %deps = (@_ == 1 ? ( $_[0] => 0 ) : @_);
+        $self->configure_requires->{ $_ } = $deps{ $_ } for keys %deps;
+    };
 
     my $file = File::Spec->catfile( getcwd(), $pl );
     eval {
