http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=541164

--- a/lib/Module/Depends.pm
+++ b/lib/Module/Depends.pm
@@ -73,15 +73,15 @@
 sub _find_modules {
     my $self = shift;
 
-    my $file = 'META.yml';
+    foreach my $file (qw/META.yml META.json/) {
     if (-e $file) {
         my $meta = LoadFile( $file );
         $self->requires( $meta->{requires} );
         $self->build_requires( $meta->{build_requires} );
+        return $self;
     }
-    else {
-        $self->error( "No META.yml found in ". $self->dist_dir );
-    }
+  }
+    $self->error( "No META.yml found in ". $self->dist_dir );
     return $self;
 }
 
