http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=541159

--- a/lib/Module/Depends.pm
+++ b/lib/Module/Depends.pm
@@ -1,6 +1,6 @@
 use strict;
 package Module::Depends;
-use YAML qw( LoadFile );
+use YAML::XS qw/LoadFile/;
 use Cwd qw( getcwd );
 use base qw( Class::Accessor::Chained );
 __PACKAGE__->mk_accessors(qw( dist_dir debug libs requires build_requires error ));
@@ -12,7 +12,6 @@
 
 =head1 SYNOPSIS
 
- use YAML;
  use Module::Depends;
  my $deps = Module::Depends->new->dist_dir( '.' )->find_modules;
  print "Our dependencies:\n", Dump $deps->requires;
