Author: Ansgar Burchardt <ansgar@43-1.org>
Subject: Adds pod to the dumpconf helper script

--- libhtml-formfu-perl.orig/bin/html_formfu_dumpconf.pl
+++ libhtml-formfu-perl/bin/html_formfu_dumpconf.pl
@@ -75,3 +75,24 @@
 Uses Config::Any and Data::Dumper to display how your config file is parsed.
 HELP
 }
+
+__END__
+
+=head1 NAME
+
+html_formfu_dumpconf.pl - dump configuration files
+
+=head1 SYNOPSIS
+
+    html_formfu_dumpconf.pl F<file.conf>
+
+    html_formfu_dumpconf.pl F<file>
+
+=head1 DESCRIPTION
+
+Uses Config::Any and Data::Dumper to display how your config file is parsed.
+
+=head1 SEE ALSO
+
+Config::Any, Data::Dumper
+
