Description: spelling mistake according to lintian
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-08-27

--- a/lib/HTML/FormFu/Manual/Cookbook.pod
+++ b/lib/HTML/FormFu/Manual/Cookbook.pod
@@ -457,7 +457,7 @@
     sub auto : Private {
         my ($self, $c) = @_;
 
-        # We want to utilize alot of the magic that the controller
+        # We want to utilize a lot of the magic that the controller
         # gives us, so therefore we call $self->form like this
 
         my $login_form = $self->form;
