Description: fix a spelling mistake
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-04-24

--- a/PGPLOT.pm
+++ b/PGPLOT.pm
@@ -223,7 +223,7 @@
 =head2 ARGUMENT MAPPING - GENERAL HANDLING OF BINARY DATA
 
 In addition to the implicit rules mentioned above PGPLOT now provides
-a scheme for explictly handling binary data in all routines.
+a scheme for explicitly handling binary data in all routines.
 
 If your scalar variable (e.g. C<$x>) holds binary data (i.e. 'packed')
 then simply pass PGPLOT a reference to it (e.g. C<\$x>). Thus one can
