Author: AGOSTINI Yves
Description: fix bad-whatis-entry warnings
--- a/lib/Module/Install/With.pm
+++ b/lib/Module/Install/With.pm
@@ -12,8 +12,13 @@
 	$ISCORE  = 1;
 }
 
+=pod
+
+=head1 NAME
 
+Module::Install::With - find environnement for Module::Install
 
+=cut
 
 
 #####################################################################
--- a/lib/Module/Install/Admin/Include.pm
+++ b/lib/Module/Install/Admin/Include.pm
@@ -9,6 +9,14 @@
 	@ISA     = qw{Module::Install::Base};
 }
 
+=pod
+
+=head1 NAME
+
+Module::Install::Admin::Include - include methods for Module::Install
+
+=cut
+
 sub include {
 	my $self = shift;
 	foreach my $rv ( $self->admin->glob_in_inc($_[0]) ) {
