Description: fix POD spelling errors
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=57528

--- a/lib/Aspect/Pointcut.pm
+++ b/lib/Aspect/Pointcut.pm
@@ -333,7 +333,7 @@
 never need to be tested.
 
 The currying process will generate and return a new pointcut tree that is
-independant from the original, and that can perform a match test at the
+independent from the original, and that can perform a match test at the
 structurally minimum computational cost.
 
 Returns a new optimised B<Aspect::Pointcut> object if any further testing
