Author: franck cuny <franck@lumberjaph.net>
Description: fix a warning about deprecated method
--- a/lib/Search/GIN/Extract/Attributes.pm
+++ b/lib/Search/GIN/Extract/Attributes.pm
@@ -33,7 +33,7 @@
     if ( $self->has_attributes ) {
         return grep { defined } map { $meta->find_attribute_by_name($_) } @{ $self->attributes };
     } else {
-        return $meta->compute_all_applicable_attributes;
+        return $meta->get_all_attributes;
     }
 }
 
