--- a/pperl.h.header
+++ b/pperl.h.header
@@ -260,9 +260,11 @@ my $DONE        = 0;   # set flag to tru
 
 # $SIG{INT} = $SIG{TERM} = sub { $DONE++ };
 
-$PPERL::SOCKET_NAME =~ m~^([a-z0-9/_-]+)$~i
+$PPERL::SOCKET_NAME =~ m~^(/.*)$~
   or die "unclean socket name '$PPERL::SOCKET_NAME'";
 $PPERL::SOCKET_NAME = $1;
+$PPERL::SOCKET_NAME =~ m~/\.\.(?:/|$)~
+  and die "unclean socket name '$PPERL::SOCKET_NAME'";
 if (-e $PPERL::SOCKET_NAME) {
   if (-e "${PPERL::SOCKET_NAME}.pid") {
     die "socket and pid file both exist - possible error state. Delete both and retry";
