Author: gregor herrmann <gregoa@debian.org>
Description: spelling fixes
Forwarded: no

--- a/etc/make_method
+++ b/etc/make_method
@@ -411,7 +411,7 @@ Read the help text for the method from t
 
 =item --code=FILE
 
-Read the actual code for the routine from the file specifed. If this option is
+Read the actual code for the routine from the file specified. If this option is
 not given, the code is read from the standard input file descriptor.
 
 =item --output=FILE
--- a/lib/RPC/XML/Server.pm
+++ b/lib/RPC/XML/Server.pm
@@ -1032,7 +1032,7 @@ closure that forces execution in the B<R
 code element of a procedure/method is passed in as a direct code reference by
 one of the other syntaxes allowed by the constructor, the package may well be
 different. Thus, routines should strive to be as localized as possible,
-independant of specific namespaces. If a group of routines are expected to
+independent of specific namespaces. If a group of routines are expected to
 work in close concert, each should explicitly set the namespace with a
 C<package> declaration as the first statement within the routines themselves.
 
