--- a/lib/Test/MockObject.pm
+++ b/lib/Test/MockObject.pm
@@ -558,8 +558,8 @@ before the actual module has a chance to
 before a use or require or place it before a C<use_ok()> or C<require_ok()>
 call.
 
-You can optionally add functions to the mocked module by passing them as name
-=> coderef pairs to C<fake_module()>.  This is handy if you want to test an
+You can optionally add functions to the mocked module by passing them as
+name => coderef pairs to C<fake_module()>.  This is handy if you want to test an
 C<import()>:
 
     my $import;
