Description: Error in POD marking.
 Insert a missing '=back' marker'
Forwarded: http://github.com/mschilli/cache-historical-perl/commit/c5f66449b188b6b5c176bf9aed64b4a4f092bb8d
Author: Mats Erik Andersson <mats.andersson@gisladisker.se>
Last-Update: 2010-02-23
--- Cache-Historical-0.03/Historical.pm.orig
+++ Cache-Historical-0.03/Historical.pm
@@ -387,6 +387,8 @@
        # update of a given key.
     my $since = $cache->since_last_update( $key );
 
+=back
+
 =head1 LEGALESE
 
 Copyright 2007-2008 by Mike Schilli, all rights reserved.
