Description: Fix some POD errors in GnuPG::Options manpage
Origin: vendor
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=50141
Forwarded: yes
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>
--- a/lib/GnuPG/Options.pm
+++ b/lib/GnuPG/Options.pm
@@ -279,6 +279,8 @@
 than once, such as B<recipients>, those are considered lists
 and passed accordingly.  Each are undefined or false to begin.
 
+=back
+
 =head2 Meta Options
 
 Meta options are those which do not correlate directly to any
@@ -297,6 +299,8 @@
 take an id; otherwise assume an object of type GnuPG::Key
 is required.
 
+=over 4
+
 =item meta_pgp_5_compatible
 
 If true, arguments are generated to try to be compatible with PGP 5.x.
