Description: fix a tiny typo in POD
Author: Damyan Ivanov <dmn@debian.org>
Forwarded: no

--- a/lib/Class/DBI/Search/Basic.pm
+++ b/lib/Class/DBI/Search/Basic.pm
@@ -70,7 +70,7 @@
 
 
 There are also a variety of private methods underneath this that could
-be overriden in a pinch, but if you need to do this I'd rather you let
+be overridden in a pinch, but if you need to do this I'd rather you let
 me know so that I can make them public, or at least so that I don't
 remove them from under your feet.
 
