Author: Joey Hess <joeyh@debian.org>

Patch for lib/Mail/Mbox/MessageParser/Config.pm
to sanitize paths.
--- a/lib/Mail/Mbox/MessageParser/Config.pm
+++ b/lib/Mail/Mbox/MessageParser/Config.pm
@@ -8,11 +8,11 @@
 
 %Mail::Mbox::MessageParser::Config = (
   'programs' => {
-    'bzip' => '/sw/bin/bzip2',
-    'bzip2' => '/sw/bin/bzip2',
+    'bzip' => '/bin/bzip2',
+    'bzip2' => '/bin/bzip2',
     'diff' => '/usr/bin/diff',
-    'grep' => undef,
-    'gzip' => '/sw/bin/gzip',
+    'grep' => '/bin/grep',
+    'gzip' => '/bin/gzip',
   },
 
   'max_testchar_buffer_size' => 1048576,
