Description: spelling mistake
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-06-30

--- a/lib/POE/Component/Server/SimpleHTTP.pm
+++ b/lib/POE/Component/Server/SimpleHTTP.pm
@@ -1610,7 +1610,7 @@
    # send a CLOSE event to the kernel with the appropriate response as parameter
    $kernel->yield('GOT_STREAM', $response);
 
-The optionnal dont_flush option gives the user the ability to control the callback
+The optional dont_flush option gives the user the ability to control the callback
 to the streaming event, which means once your stream event has reached its end
 it won't be called, you have to call it back.
 
@@ -1632,7 +1632,7 @@
       #
       # $kernel->delay('GOT_STREAM', 1, $stream );
 
-      # otherwise the GOT_STREAM event is triggered continously until
+      # otherwise the GOT_STREAM event is triggered continuously until
       # we call the CLOSE event on the response like that :
       #
       if ($heap{'streaming_is_done'}) {
