Description: fix POD spelling errors
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: no
--- a/lib/Net/SFTP/Foreign.pm
+++ b/lib/Net/SFTP/Foreign.pm
@@ -3065,7 +3065,7 @@
 
 Well, both modules have their pros and cons:
 
-Net::SFTP::Foreign does not requiere a bunch of additional modules and
+Net::SFTP::Foreign does not require a bunch of additional modules and
 external libraries to work, just the OpenBSD SSH client (or any other
 client compatible enough).
 
@@ -3640,7 +3640,7 @@
 
 =item wanted =E<gt> qr/.../
 
-Only elements which filename match the regular expresion are included
+Only elements which filename match the regular expression are included
 on the listing.
 
 =item wanted =E<gt> sub {...}
@@ -3767,7 +3767,7 @@
 
 =item ordered =E<gt> 1
 
-By default, the file system is searched in an implementation dependant
+By default, the file system is searched in an implementation dependent
 order (actually optimized for low memory comsumption). If this option
 is included, the file system is searched in a deep-first, sorted by
 filename fashion.
@@ -3851,7 +3851,7 @@
 
 =item strict_leading_dot =E<gt> 0
 
-by default, a dot character at the begining of a file or directory
+by default, a dot character at the beginning of a file or directory
 name is not matched by willcards (C<*> or C<?>). Setting this flags to
 a false value changes this behaviour.
 
@@ -4232,7 +4232,7 @@
 =item $sftp-E<gt>opendir($path)
 
 Sends a C<SSH_FXP_OPENDIR> command to open the remote directory
-C<$path>, and returns an open handle on success (unfortunatelly,
+C<$path>, and returns an open handle on success (unfortunately,
 current versions of perl does not support directory operations via
 tied handles, so it is not possible to use the returned handle as a
 native one).
