Author: Nicholas Bamber <nicholas@periapt.co.uk>
Forwarded: no
Last-Updated: 2011-07-08
Subject: spelling
--- a/lib/Net/SFTP/Foreign.pm
+++ b/lib/Net/SFTP/Foreign.pm
@@ -3274,7 +3274,7 @@
 knows nothing about the character encoding used on the remote
 filesystem to represent file and directory names.
 
-This option allows to select the encoding used in the remote
+This option allows one to select the encoding used in the remote
 machine. The default value is C<utf8>.
 
 For instance:
@@ -3326,7 +3326,7 @@
 
 =item transport =E<gt> [$in_fh, $out_fh, $pid]
 
-allows to use an already open pipe or socket as the transport for the
+allows one to use an already open pipe or socket as the transport for the
 SFTP protocol.
 
 It can be (ab)used to make this module work with password
@@ -3344,7 +3344,7 @@
 
 =item open2_cmd =E<gt> $cmd;
 
-allows to completely redefine how C<ssh> is called. Its arguments are
+allows one to completely redefine how C<ssh> is called. Its arguments are
 passed to L<IPC::Open2::open2> to open a pipe to the remote
 server.
 
@@ -3418,7 +3418,7 @@
 that results on the SSH connection being closed by the first process
 where the object goes out of scope, something undesirable.
 
-This option allows to work-around this issue to some extend.
+This option allows one to work-around this issue to some extend.
 
 The acceptable values for C<$ad> are:
 
@@ -3542,7 +3542,7 @@
 
 =item umask =E<gt> $umask
 
-allows to select the umask to apply when setting the permissions of
+allows one to select the umask to apply when setting the permissions of
 the copied file. Default is to use the umask for the current process.
 
 =item perm =E<gt> $perm
@@ -3635,7 +3635,7 @@
 =item queue_size =E<gt> $size
 
 read and write requests are pipelined in order to maximize transfer
-throughput. This option allows to set the maximum number of requests
+throughput. This option allows one to set the maximum number of requests
 that can be concurrently waiting for a server response.
 
 =back
@@ -3689,7 +3689,7 @@
 
 =item umask =E<gt> $umask
 
-allows to select the umask to apply when setting the permissions of
+allows one to select the umask to apply when setting the permissions of
 the copied file. Default is to use the umask for the current process.
 
 =item perm =E<gt> $perm
@@ -3747,7 +3747,7 @@
 =item queue_size =E<gt> $size
 
 read and write requests are pipelined in order to maximize transfer
-throughput. This option allows to set the maximum number of requests
+throughput. This option allows one to set the maximum number of requests
 that can be concurrently waiting for a server response.
 
 =item late_set_perm =E<gt> $bool
@@ -4105,7 +4105,7 @@
 
 =item no_wanted =E<gt> ...
 
-This option allows to select which files and directories have to be
+This option allows one to select which files and directories have to be
 copied. See also C<ls> method docs.
 
 If a directory is discarded all of its contents are also discarded (as
@@ -4173,7 +4173,7 @@
 
 =item no_wanted =E<gt> ...
 
-This option allows to select which files and directories have to be
+This option allows one to select which files and directories have to be
 copied. See also C<ls> method docs.
 
 If a directory is discarded all of its contents are also discarded (as
@@ -4324,7 +4324,7 @@
 
 =back
 
-When creating a new remote file, C<$attrs> allows to set its initial
+When creating a new remote file, C<$attrs> allows one to set its initial
 attributes. C<$attrs> has to be an object of class
 L<Net::SFTP::Foreign::Attributes>.
 
@@ -4734,7 +4734,7 @@
      or die "unable to transfer file: " . $sftp->error;
 
 Some servers do not support the C<fsetstat> method on open file
-handles. Setting this flag allows to delay that operation until the
+handles. Setting this flag allows one to delay that operation until the
 file has been completely transferred and the remote file handle
 closed.
 
@@ -4896,13 +4896,13 @@
 Net::SFTP::Foreign integrates nicely with my other module
 L<Net::OpenSSH>.
 
-L<Net::SFTP::Foreign::Backend::Net_SSH2> allows to run
+L<Net::SFTP::Foreign::Backend::Net_SSH2> allows one to run
 Net::SFTP::Foreign on top of L<Net::SSH2>.
 
 Modules offering similar functionality available from CPAN are
 L<Net::SFTP> and L<Net::SSH2>.
 
-L<Test::SFTP> allows to run tests against a remote SFTP server.
+L<Test::SFTP> allows one to run tests against a remote SFTP server.
 
 L<autodie>.
 
