Description: fix POD spelling errors
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: no
--- a/lib/Config/YAML.pm
+++ b/lib/Config/YAML.pm
@@ -77,7 +77,7 @@
 creation. It is required, and must be the first parameter given. If
 the second parameter is C<output>, then it is used to specify the file
 to which configuration data will later be written out.  This
-positional dependancy makes it possible to have parameters named
+positional dependency makes it possible to have parameters named
 "config" and/or "output" in config files.
 
 Initial configuration values can be passed as subsequent parameters to
