Author: gregor herrmann <gregoa@debian.org>
Description: fix some minor spelling mistakes
Forwarded: no

--- a/lib/DBI/FAQ.pm
+++ b/lib/DBI/FAQ.pm
@@ -66,7 +66,7 @@ To quote Tim Bunce, the architect and au
     ``DBI is a database access Application Programming Interface (API)
       for the Perl Language. The DBI API Specification defines a set
       of functions, variables and conventions that provide a consistent
-      database interface independant of the actual database being used.''
+      database interface independent of the actual database being used.''
 
 In simple language, the DBI interface allows users to access multiple database
 types transparently. So, if you connecting to an Oracle, Informix, mSQL, Sybase
--- a/lib/DBI/ProxyServer.pm
+++ b/lib/DBI/ProxyServer.pm
@@ -604,7 +604,7 @@ every workstation shall be able to execu
 There is a perl program "dbiproxy" which runs on a machine which is able
 to connect to all the databases we wish to reach. All Perl-DBD-drivers must
 be installed on this machine. You can also reach databases for which drivers 
-are not available on the machine where you run the programm querying the 
+are not available on the machine where you run the program querying the
 database, e.g. ask MS-Access-database from Linux.
 
 Create a configuration file "proxy_oracle.cfg" at the dbproxy-server:
@@ -651,7 +651,7 @@ Create a configuration file "proxy_oracl
 		},
 
 		# rule: employee_workplace
-		# desc: get detailled informations
+		# desc: get detailed information
 		{
 			# any IP-address is meant here
 			mask => '^10\.95\.81\.(\d+)$',
@@ -687,7 +687,7 @@ Start the proxyserver like this:
 
 =head2 Testing the connection from a remote machine
 
-Call a programm "dbish" from your commandline. I take the machine from rule "internal_webserver"
+Call a program "dbish" from your commandline. I take the machine from rule "internal_webserver"
 
 	dbish "dbi:Proxy:hostname=oracle.zdf;port=12400;dsn=dbi:Oracle:e01" informationdesk xxx
 
