Description: fix POD spelling error
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: no
--- a/lib/SVN/Dump/Reader.pm
+++ b/lib/SVN/Dump/Reader.pm
@@ -236,7 +236,7 @@
 If the option C<check_digest> is enabled, this method will recompute
 the digests for a dump without deltas, based on the information in the
 C<Text-content-md5> and C<Text-content-sha1> headers (if the corresponding
-C<Digest::> module is availabled). In case of a mismatch, the routine
+C<Digest::> module is available). In case of a mismatch, the routine
 will C<die()> with an exception complaining about a C<checksum mismatch>.
 
 =item read_header_block( )
