Create a way to disable tests that require Internet connectivity.
Index: libwww-mechanize-perl-1.52/t/live/wikipedia.t
===================================================================
--- libwww-mechanize-perl-1.52.orig/t/live/wikipedia.t	2007-10-25 09:34:52.000000000 -0700
+++ libwww-mechanize-perl-1.52/t/live/wikipedia.t	2008-12-18 10:15:46.000000000 -0800
@@ -3,6 +3,10 @@
 use warnings;
 use strict;
 
+if ($ENV{NOINTERNET} ) {
+	use Test::More skip_all => "We don't have internet here";
+}
+
 use constant LANGUAGES => qw( en it ja es nl pl );
 use Test::More tests => 3 + (2 * scalar LANGUAGES);
 
Index: libwww-mechanize-perl-1.52/t/live/encoding.t
===================================================================
--- libwww-mechanize-perl-1.52.orig/t/live/encoding.t	2008-12-18 10:20:40.000000000 -0800
+++ libwww-mechanize-perl-1.52/t/live/encoding.t	2008-12-18 10:20:58.000000000 -0800
@@ -3,6 +3,10 @@
 use strict;
 use warnings;
 
+if ($ENV{NOINTERNET} ) {
+	use Test::More skip_all => "We don't have internet here" ; 
+}
+
 use constant PAIRS => {
     'http://delicious.com/'
         => 'utf-8',
