Description: fix POD spelling errors
Origin: vendor
Forwarded: no
Author: Jonathan Yu <jawnsy@cpan.org>

--- a/lib/Pod/Simple/XHTML.pm
+++ b/lib/Pod/Simple/XHTML.pm
@@ -151,7 +151,7 @@
 A string containing all characters that should be encoded as HTML entities,
 specified using the regular expression character class syntax (what you find
 within brackets in regular expressions). This value will be passed as the
-second argument to the C<encode_entities> fuction of L<HTML::Entities>. IF
+second argument to the C<encode_entities> function of L<HTML::Entities>. IF
 L<HTML::Entities> is not installed, then any characters other than C<&<>"'>
 will be encoded numerically.
 
