Description: Fix spelling error
Author: Fabrizio Regalli <fabreg@fabreg.it>
Last-Update: 2011-07-15
Forwarded: no

--- a/ReadBackwards.pm
+++ b/ReadBackwards.pm
@@ -292,7 +292,7 @@
 =head2 getline
 
 C<getline> is an alias for the readline method. It is here for
-compatibilty with the IO::* classes which has a getline method.
+compatibility with the IO::* classes which has a getline method.
 
 =head2 eof
 
