Author: Florian Ragwitz <rafl@debian.org>
Description: trailing newline for HandleConsumer and FileConsumer
Index: libxml-sax-writer-perl/t/10consumers.t
===================================================================
--- libxml-sax-writer-perl.orig/t/10consumers.t	2009-02-25 00:52:04.000000000 +0000
+++ libxml-sax-writer-perl/t/10consumers.t	2009-02-25 22:57:37.000000000 +0000
@@ -71,8 +71,8 @@
 my $cnt2 = <FH2>;
 close FH2;
 
-ok($cnt1 eq 'FILE ONE FILE ONE', 'file content (1)');
-ok($cnt2 eq 'FILE TWO FILE TWO', 'file content (2)');
+ok($cnt1 eq "FILE ONE FILE ONE\n", 'file content (1)');
+ok($cnt2 eq "FILE TWO FILE TWO\n", 'file content (2)');
 
 
 ##
Index: libxml-sax-writer-perl/lib/XML/SAX/Writer.pm
===================================================================
--- libxml-sax-writer-perl.orig/lib/XML/SAX/Writer.pm	2009-02-25 00:53:16.000000000 +0000
+++ libxml-sax-writer-perl/lib/XML/SAX/Writer.pm	2009-02-25 23:00:41.000000000 +0000
@@ -100,7 +100,7 @@
             $ref eq 'GLOB'                                or
             UNIVERSAL::isa(\$self->{Output}, 'GLOB')      or
             UNIVERSAL::isa($self->{Output}, 'IO::Handle')) {
-        $self->{Consumer} = XML::SAX::Writer::HandleConsumer->new($self->{Output});
+        $self->{Consumer} = XML::SAX::Writer::HandleConsumer->new($self->{Output}, $self);
     }
     elsif (not $ref) {
         $self->{Consumer} = XML::SAX::Writer::FileConsumer->new($self->{Output});
@@ -315,7 +315,11 @@
 #-------------------------------------------------------------------#
 # finalize
 #-------------------------------------------------------------------#
-sub finalize { return 0 }
+sub finalize {
+    my $fh = ${$_[0]};
+    print $fh "\n";
+    return 0
+}
 #-------------------------------------------------------------------#
 
 
@@ -350,7 +354,9 @@
 # finalize
 #-------------------------------------------------------------------#
 sub finalize {
-    close ${$_[0]};
+    my $fh = ${$_[0]};
+    print $fh "\n";
+    close $fh;
     return 0;
 }
 #-------------------------------------------------------------------#
