adding a missing dependency to a test
Index: libxml-sax-writer-perl/t/05basic.t
===================================================================
--- libxml-sax-writer-perl.orig/t/05basic.t	2009-02-25 00:53:16.000000000 +0000
+++ libxml-sax-writer-perl/t/05basic.t	2009-02-25 23:11:30.000000000 +0000
@@ -7,6 +7,7 @@
 
 use strict;
 use Test::More tests => 29;
+use Text::Iconv;
 BEGIN { use_ok('XML::SAX::Writer'); }
 
 # VMS has different names for codepages
