Author: Frank Wiegand <fwie@cpan.org>
Origin: http://rt.cpan.org/Ticket/Attachment/692202/356379/diff
Bug: http://rt.cpan.org/Ticket/Display.html?id=51618
Subject: POD nits [ C<...> construct missing left side ]
Last-Update: 2010-08-22
--- a/lib/XML/SAX/Writer.pm
+++ b/lib/XML/SAX/Writer.pm
@@ -635,17 +635,17 @@
 
 This is used to set the proper converter for character encodings. The
 default implementation should suffice but you can override it. It must
-set C<$self->{Encoder}> to an Encoder object. Subclasses *should* call
+set C<< $self->{Encoder} >> to an Encoder object. Subclasses *should* call
 it.
 
 =item setConsumer
 
 Same as above, except that it is for the Consumer object, and that it
-must set C<$self->{Consumer}>.
+must set C<< $self->{Consumer} >>.
 
 =item setEscaperRegex
 
-Will initialise the escaping regex C<$self->{EscaperRegex}> based on
+Will initialise the escaping regex C<< $self->{EscaperRegex} >> based on
 what is needed.
 
 =item escape STRING
