Author: gregor herrmann <gregoa@debian.org>
Description: we want the modules in /usr/share/perl5

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -40,8 +40,8 @@
                            'Test::More' => '0.62'
                          },
           'EXE_FILES'    => [],
-          'INSTALLDIRS'  => ($] >= 5.008009) ? "perl" : "site",
-          'INST_LIB'     => ($] >= 5.008009) ? 'blib/arch' : 'blib/lib',
+#          'INSTALLDIRS'  => ($] >= 5.008009) ? "perl" : "site",
+#          'INST_LIB'     => ($] >= 5.008009) ? 'blib/arch' : 'blib/lib',
           'PL_FILES'     => {},
           'realclean'    => {FILES=> join ' ', @files},
           # In the core pods will be built by installman.
