From: Ansgar Burchardt <ansgar@43-1.org>
Date: Sat, 09 May 2009 19:58:44 +0200
Subject: Fix POD errors

--- libhtml-formfu-model-dbic-perl.orig/lib/HTML/FormFu/Model/DBIC.pm
+++ libhtml-formfu-model-dbic-perl/lib/HTML/FormFu/Model/DBIC.pm
@@ -1400,6 +1400,8 @@
 The field's L<name|HTML::FormFu::Element::_Field/name> must be set to the 
 name of the C<many_to_many> relationship.
 
+=over
+
 =item default_column
 
 If you want to search / associate the related table by a column other it's
@@ -1423,6 +1425,7 @@
             link_values:
               foo: bar
 
+=back
 
 The default implementation will first remove all related objects and set the
 new ones (see L<http://search.cpan.org/perldoc?DBIx::Class::Relationship::Base#set_$rel>).
