Description: fix POD spelling
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=54388
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=54388
Author: Jonathan Yu <jawnsy@cpan.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-06-13

--- libnamespace-clean-perl.orig/lib/namespace/clean.pm
+++ libnamespace-clean-perl/lib/namespace/clean.pm
@@ -227,9 +227,9 @@
 If you just want to C<-except> a single sub, you can pass it directly.
 For more than one value you have to use an array reference.
 
-=head2 Explicitely removing functions when your scope is compiled
+=head2 Explicitly removing functions when your scope is compiled
 
-It is also possible to explicitely tell C<namespace::clean> what packages
+It is also possible to explicitly tell C<namespace::clean> what packages
 to remove when the surrounding scope has finished compiling. Here is an
 example:
 
