Author: gregor herrmann <gregoa@debian.org>
Description: spelling mistake
Forwarded: no
Origian: vendor

--- a/lib/Net/GPSD3.pm
+++ b/lib/Net/GPSD3.pm
@@ -33,7 +33,7 @@
 
 =head1 DESCRIPTION
 
-Net::GPSD3 provides an object client interface to the gpsd server daemon utilizing the version 3.1 protocol. gpsd is an open source GPS deamon from http://gpsd.berlios.de/.  Support for Version 3 of the protocol (JSON) was adding to the daemon in version 2.90.  If your daemon is before 2.90 then please use the L<Net::GPSD> package.
+Net::GPSD3 provides an object client interface to the gpsd server daemon utilizing the version 3.1 protocol. gpsd is an open source GPS daemon from http://gpsd.berlios.de/.  Support for Version 3 of the protocol (JSON) was adding to the daemon in version 2.90.  If your daemon is before 2.90 then please use the L<Net::GPSD> package.
 
 =head1 CONSTRUCTOR
 
