Description: fix POD spelling errors
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: no
--- a/lib/Image/Size.pm
+++ b/lib/Image/Size.pm
@@ -760,7 +760,7 @@
 file handles and scalar references cannot be reliably transformed into a
 unique key for the table of cache data. Buffers could be cached using the
 MD5 module, and perhaps in the future I will make that an option. I do not,
-however, wish to lengthen the dependancy list by another item at this time.
+however, wish to lengthen the dependency list by another item at this time.
 
 As B<Image::Magick> operates on file names, not handles, the use of it is
 restricted to cases where the input to C<imgsize> is provided as file name.
