--- a/Pod/FindPods.pm
+++ b/Pod/FindPods.pm
@@ -549,7 +549,7 @@
 
 =head1 AUTHOR
 
-Slaven Reziæ <F<slaven@rezic.de>>
+Slaven ReziE<0x107> <F<slaven@rezic.de>>
 
 Copyright (c) 2001,2003,2004,2005 Slaven Rezic. All rights reserved.
 This program is free software; you can redistribute it and/or modify
--- a/Pod/Search_db.pm
+++ b/Pod/Search_db.pm
@@ -223,7 +223,7 @@
 Most of the code here is borrowed from L<perlindex> written by
 Ulrich Pfeifer <F<Ulrich.Pfeifer@de.uu.net>>.
 
-Current maintainer is Slaven Reziæ <F<slaven@rezic.de>>.
+Current maintainer is Slaven Rezic <F<slaven@rezic.de>>.
 
 Copyright (c) 1997-1998 Achim Bohnet. All rights reserved.  This program is
 free software; you can redistribute it and/or modify it under the same
--- a/Pod/Search.pm
+++ b/Pod/Search.pm
@@ -320,7 +320,7 @@
 
 Achim Bohnet <F<ach@mpe.mpg.de>>
 
-Current maintainer is Slaven Reziæ <F<slaven@rezic.de>>.
+Current maintainer is Slaven ReziE<0x107> <F<slaven@rezic.de>>.
 
 Copyright (c) 1997-1998 Achim Bohnet. All rights reserved.  This program
 is free software; you can redistribute it and/or modify it under the same
--- a/Pod/SimpleBridge.pm
+++ b/Pod/SimpleBridge.pm
@@ -427,6 +427,6 @@
 the old Tk::Pod::Text code that Nick Ing-Simmons
 <F<nick@ni-s.u-net.com>> originally wrote.
 
-Current maintainer is Slaven Reziæ <F<slaven@rezic.de>>.
+Current maintainer is Slaven ReziE<0x107> <F<slaven@rezic.de>>.
 
 =cut
