Fixed problem translating from pod to man
--- a/lib/Net/BitTorrent.pm
+++ b/lib/Net/BitTorrent.pm
@@ -1162,7 +1162,7 @@
 
 =back
 
-=head2 Net::BitTorrent::Torrent::Tracker::HTTP/Net::BitTorrent::Torrent::Tracker::UDP
+=head2 Net::BitTorrent::Torrent::Tracker::HTTP / Net::BitTorrent::Torrent::Tracker::UDP
 
 Note: The tracker objects passed to these callbacks will either be a
 L<Net::BitTorrent::Torrent::Tracker::HTTP|Net::BitTorrent::Torrent::Tracker::HTTP>
