Description: Disable tests which require a network connection
Forwarded: not-needed
Author: Florian Ragwitz <rafl@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-08-08

--- a/t/2_client_jabberd1.4.t
+++ b/t/2_client_jabberd1.4.t
@@ -1,5 +1,5 @@
 use lib "t/lib";
-use Test::More tests=>5;
+use Test::More skip_all => 'Disabled for the Debian build';
 
 BEGIN{ use_ok( "Net::XMPP" ); }
 
--- a/t/3_client_jabberd2.t
+++ b/t/3_client_jabberd2.t
@@ -1,5 +1,5 @@
 use lib "t/lib";
-use Test::More tests=>5;
+use Test::More skip_all => 'Disabled for the Debian build';
 
 BEGIN{ use_ok( "Net::XMPP" ); }
 
