Description: fix POD whatis entries
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: no
--- a/lib/SQL/Translator/Diff.pm
+++ b/lib/SQL/Translator/Diff.pm
@@ -424,7 +424,7 @@
 
 =head1 NAME
 
-SQL::Translator::Diff
+SQL::Translator::Diff - determine differences between two schemas
 
 =head1 DESCRIPTION
 
--- a/lib/SQL/Translator/Manual.pod
+++ b/lib/SQL/Translator/Manual.pod
@@ -1,6 +1,6 @@
 =head1 NAME
 
-SQL::Translator::Manual
+SQL::Translator::Manual - sqlfairy user manual
 
 =head1 SYNOPSIS
 
