Description: fix spelling-error-in-manpage
Forwarded: https://rt.cpan.org/Public/Bug/Display.html?id=70788

--- a/lib/MIME/Lite.pm
+++ b/lib/MIME/Lite.pm
@@ -2551,7 +2551,7 @@
 All of your $msg-E<gt>send invocations will work as expected.
 
 After sending, the method last_send_successful() can be used to determine
-if the send was succesful or not.
+if the send was successful or not.
 
 =cut
 
@@ -2652,7 +2652,7 @@
 =back
 
 After sending, the method last_send_successful() can be used to determine
-if the send was succesful or not.
+if the send was successful or not.
 
 =cut
 
@@ -2816,7 +2816,7 @@
 True on success, croaks with an error message on failure.
 
 After sending, the method last_send_successful() can be used to determine
-if the send was succesful or not.
+if the send was successful or not.
 
 =cut
 
@@ -3613,7 +3613,7 @@
 form, generally they are documented, but they should be easy to understand.
 
 The ./contrib directory contains a companion/tool modules that come bundled
-with MIME::Lite, they dont get installed by default. Please review the POD they
+with MIME::Lite, they don't get installed by default. Please review the POD they
 come with.
 
 =head1 BUGS
@@ -3634,7 +3634,7 @@
 of not using Test::More that I can see.
 
 Bug fixes / Patches / Contribution are welcome, however I probably won't apply them
-unless they also have an associated test. This means that if I dont have the time to
+unless they also have an associated test. This means that if I don't have the time to
 write the test the patch wont get applied, so please, include tests for any patches
 you provide.
 
