Description: Fix POD errors
Forwarded: http://mail-archives.apache.org/mod_mbox/perl-modperl/201004.mbox/thread
Author: Damyan Ivanov <dmn@debian.org>
Last-Update: 2011-03-22
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/Apache-Test/lib/Apache/Test.pm
+++ b/Apache-Test/lib/Apache/Test.pm
@@ -1020,6 +1020,8 @@
 
 =head1 Additional Configuration Variables
 
+=over
+
 =item basic_config
 
   my $basic_cfg = Apache::Test::basic_config();
@@ -1052,6 +1054,8 @@
 returned. If one or more arguments are passed the corresponding values
 are returned.
 
+=back
+
 =head1 Test::More Integration
 
 There are a few caveats if you want to use I<Apache::Test> with
