Description: fix spelling errors
Origin: vendor
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=71254
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-03-11

--- a/lib/perl5i.pm
+++ b/lib/perl5i.pm
@@ -509,7 +509,7 @@
 Turns a number like 1234567 into a string like 1,234,567 known as "digit grouping".
 
 It honors your current locale to determine the separator and grouping.
-This can be overriden using C<%options>.
+This can be overridden using C<%options>.
 
 NOTE: many systems do not have their numeric locales set properly
 
@@ -555,7 +555,7 @@
 locale settings.
 
 C<%options> defaults to C<< ( separator => ",", grouping => 3, decimal_point => "." ) >>.
-Each key will be overriden individually.
+Each key will be overridden individually.
 
     1234->commify;                      # 1,234
     1234->commify({ separator => "." });  # 1.234
--- a/lib/perl5i/Meta.pod
+++ b/lib/perl5i/Meta.pod
@@ -35,8 +35,8 @@
 
 =head2 Meta Instance vs Meta Class
 
-Each object has a meta object for their instance, accessable with C<<
-$obj->mo >> and also a meta object for their class, accessable with
+Each object has a meta object for their instance, accessible with C<<
+$obj->mo >> and also a meta object for their class, accessible with
 C<< $obj->mc >>.  The meta instance can do most everything the meta
 class can, mc is provided mostly for disambiguation.
 
--- a/lib/perl5ifaq.pod
+++ b/lib/perl5ifaq.pod
@@ -339,7 +339,7 @@
 
 When depending on perl5i, depend on the specific major version.  That
 is, depend on C<perl5i::2> and not C<perl5i>.  This is because older
-versions will eventually be spun out into their own seperate
+versions will eventually be spun out into their own separate
 distributions to avoid cluttering the main dist.  If you depend on
 C<perl5i::2> then the CPAN shell will always be able to find it.
 
