Description: fix POD errors
Author: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: no
--- a/lib/WWW/Selenium.pm
+++ b/lib/WWW/Selenium.pm
@@ -3213,7 +3213,7 @@
 
 "default" - The default library.  Currently the default library is "ajaxslt" .
 
- =back
+=back
 
  If libraryName isn't one of these three, then  no change will be made.
 
@@ -3421,7 +3421,7 @@
 }
 
 
-=item * $sel-E<gt>is_location($expected_location)
+=item $sel-E<gt>is_location($expected_location)
 
 Verify the location of the current page ends with the expected location.
 If an URL querystring is provided, this is checked as well.
@@ -3445,7 +3445,7 @@
     return $loc =~ /\Q$expected_location\E$/;
 }
 
-=item * $sel-E<gt>get_checked($locator)
+=item $sel-E<gt>get_checked($locator)
 
 Gets whether a toggle-button (checkbox/radio) is checked.  Fails if the specified element doesn't exist or isn't a toggle-button.
 
@@ -3466,7 +3466,7 @@
     return $self->get_string("isChecked", @_) ? 'true' : 'false';
 }
 
-=item * $sel-E<gt>is_selected($locator, $option_locator)
+=item $sel-E<gt>is_selected($locator, $option_locator)
 
 Verifies that the selected option of a drop-down satisfies the optionSpecifier.
 
@@ -3496,7 +3496,7 @@
     return grep { $pattern eq $_ } @selected;
 }
 
-=item * $sel-E<gt>get_selected_options($locator)
+=item $sel-E<gt>get_selected_options($locator)
 
 Gets all option labels for selected options in the specified select or multi-select element.
 
@@ -3517,7 +3517,7 @@
     return $self->get_string_array("getSelectedLabels", @_);
 }
 
-=item * $sel-E<gt>get_absolute_location()
+=item $sel-E<gt>get_absolute_location()
 
 Gets the absolute URL of the current page.
 
