From: Ansgar Burchardt <ansgar@43-1.org>
Date: Fri, 28 May 2010 01:04:53 +0900
Subject: Fix spelling error in documentation
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=57950

--- version-0.82.orig/lib/version.pod
+++ version-0.82/lib/version.pod
@@ -59,7 +59,7 @@
 =item Dotted Decimal Versions
 
 The more modern form of version assignment, with 3 (or potentially more)
-integers seperated by decimal points (e.g. v1.2.3).  This is the form that
+integers separated by decimal points (e.g. v1.2.3).  This is the form that
 Perl itself has used since 5.6.0 was released.  The leading "v" is now
 strongly recommended for clarity, and will throw a warning in a future
 release if omitted.
