Author: Niko Tyni <ntyni@debian.org>
Description: FTBFS: Test failure with CGI.pm >= 3.38
Bug: #521971

--- a/Embperl.pm
+++ b/Embperl.pm
@@ -319,7 +319,7 @@
 	{
     	# the param_fetch needs CGI.pm 2.43
 	#$params = $cgi->param_fetch( $_ ) ;
-    	$params = $cgi->{$_} ;
+    	$params = $CGI::VERSION >= 2.43 ? $cgi->param_fetch( $_ ) : $cgi->{$_} ;
 	if ($#$params > 0)
 	    {
 	    $fdat->{ $_ } = join ("\t", @$params) ;
