From: Niko Tyni <ntyni@debian.org>
Bug: #468237
Subject: FTBFS with Perl 5.10 (failed tests)


--- a/test.pl
+++ b/test.pl
@@ -45,6 +45,7 @@
 # input_escmode =>
 # portadd =>
 
+my $newperl = ($] >= 5.009 ? 1 : 0);
 @testdata = (
     'ascii' => { },
     'pure.htm' => {
@@ -62,25 +63,25 @@
         },
     'error.htm' => { 
         'repeat'     => 3,
-        'errors'     => 8,
+        'errors'     => 8+$newperl,
         'version'    => 1,
         },
     'error.htm' => { 
         'repeat'     => 3,
-        'errors'     => 5,
+        'errors'     => 5+$newperl,
         'version'    => 2,
         'cgi'        => 0,
         },
     'error.htm' => { 
         'repeat'     => 3,
-        'errors'     => 5,
+        'errors'     => 5+$newperl,
         'version'    => 2,
         'cgi'        => 1,
         'condition'  => '!$MP2',
         },
     'error.htm' => { 
         'repeat'     => 3,
-        'errors'     => 6,
+        'errors'     => 6+$newperl,
         'version'    => 2,
         'cgi'        => 1,
         'condition'  => '$MP2',
@@ -110,19 +111,19 @@
         },
     'errdoc/errdoc.htm' => { 
         'option'     => '262144',
-        'errors'     => 6,
+        'errors'     => 6+$newperl,
         'version'    => 1,
         'cgi'        => 0,
         },
     'errdoc/errdoc.htm' => { 
         'option'     => '262144',
-        'errors'     => 5,
+        'errors'     => 5+$newperl,
         'version'    => 2,
         'modperl'    => 1,
         },
     'errdoc/epl/errdoc2.htm' => { 
         'option'     => '262144',
-        'errors'     => 6,
+        'errors'     => 6+$newperl,
         'version'    => 1,
         'cgi'        => 0,
         'noloop'     => 1,
@@ -130,7 +131,7 @@
         },
     'errdoc/epl/errdoc2.htm' => { 
         'option'     => '262144',
-        'errors'     => 5,
+        'errors'     => 5+$newperl,
         'version'    => 2,
         'cgi'        => 0,
         'noloop'     => 1,
@@ -2264,7 +2265,7 @@
                 				}) ;
 		$t_exec += 0 ; # Embperl::Clock () - $t1 ; 
 		    
-                $err = CheckError ($EP2?5:8) if ($err == 0) ;
+                $err = CheckError (($EP2?5:8)+$newperl) if ($err == 0) ;
 
                 if (@errors != ($EP2?4:12))
                     {
--- a/test/cmp/errdoc2.htm
+++ b/test/cmp/errdoc2.htm
@@ -24,7 +24,8 @@
 ^syntax error at 
 ^syntax error at 
 ^syntax error at 
-^}&quot;</td></tr>
+^}&quot;(</td></tr>)?
+^-Missing right curly or square bracket at .*?errdoc2.htm line \d+, at end of line</td></tr>
 </table>
 
 </body>
--- a/test/cmp/error.htm
+++ b/test/cmp/error.htm
@@ -32,6 +32,7 @@
 ^<br>&nbsp;&nbsp;&nbsp;&nbsp;                                
 ^-<br>&nbsp;&nbsp;&nbsp;&nbsp;                                
 ^-<br>&nbsp;&nbsp;&nbsp;&nbsp;}
+^-<br>&nbsp;&nbsp;&nbsp;&nbsp;Missing right curly or square bracket at .*?error.htm line \d+, at end of line
 <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 </td></tr>
 </table>
