From: Ansgar Burchardt <ansgar@43-1.org>
Date: Sun, 06 Jun 2010 21:40:08 +0900
Origin: vendor
Subject: Use is_deeply correctly

--- libmodule-info-perl.orig/t/Module-Info.t
+++ libmodule-info-perl/t/Module-Info.t
@@ -290,6 +290,6 @@
     $module = Module::Info->new_from_file('t/lib/Bar.pm');
     @mods   = $module->modules_used;
     is( @mods, 3, 'modules_used with complex BEGIN block' );
-    is_deeply( sort @mods,
-               (sort qw(Cwd Carp strict)) );
+    is_deeply( [sort @mods],
+               [sort qw(Cwd Carp strict)] );
 }
