From: Ansgar Burchardt <ansgar@43-1.org>
Date: Sun, 06 Jun 2010 21:35:17 +0900
Origin: vendor
Subject: Compare versions as numbers, not as strings

--- libmodule-info-perl.orig/t/n3_version.t
+++ libmodule-info-perl/t/n3_version.t
@@ -20,18 +20,18 @@
 }
 
 is( $moo->use_version, 0 );
-is( $moo->version, '0.12' );
-is( $boo->version, '1.35', 'proper quoting in complex expression' );
+is( $moo->version, 0.12 );
+is( $boo->version, 1.35, 'proper quoting in complex expression' );
 
 SKIP: {
     skip 'version.pm not found', 5 unless $has_version_pm;
 
     is( $moo_ver->use_version, 1 );
     isa_ok( $moo_ver->version, 'version' );
-    is( $moo_ver->version, '0.120' );
+    is( $moo_ver->version, 0.120 );
 
     isa_ok( $foo_ver->version, 'version' );
-    is( $foo_ver->version, '7.254' );
+    is( $foo_ver->version, 7.254 );
 }
 
 SKIP: {
