diff -urNad libclass-dbi-loader-relationship-perl-1.2~/Relationship.pm libclass-dbi-loader-relationship-perl-1.2/Relationship.pm
--- libclass-dbi-loader-relationship-perl-1.2~/Relationship.pm	2004-03-01 16:48:45.000000000 +0000
+++ libclass-dbi-loader-relationship-perl-1.2/Relationship.pm	2005-10-12 09:51:07.000000000 +0100
@@ -93,8 +93,8 @@
     return "$from_c->$method($mapper => $to_c);\n".
            ($method ne "has_a" && "$to_c->has_a(".to_S($from)." => $from_c);\n") 
            if $DEBUG;
-    $from_c->$method($mapper => $to_c);
     $to_c->has_a(to_S($from) => $from_c) unless $method eq "has_a";
+    $from_c->$method($mapper => $to_c);
 }
 
 1;
