Author: gregor herrmann <gregoa@debian.org>
Description: spelling fix
Forwarded: no
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
Last-Update: 2011-03-18

--- a/lib/NetPacket/UDP.pm
+++ b/lib/NetPacket/UDP.pm
@@ -309,7 +309,7 @@
             # replace foo in the payload with bar
             $udp_obj->{data} =~ s/foo/bar/g;
 
-            # reencode the packet
+            # re-encode the packet
             $ip_obj->{data} = $udp_obj->encode($ip_obj);
             $data = $ip_obj->encode;
 
