From: Ansgar Burchardt <ansgar@43-1.org>
Date: Thu, 15 Jul 2010 21:08:28 +0900
Origin: vendor
Subject: Add whatis entry to documentation

--- libmoosex-semiaffordanceaccessor-perl.orig/lib/MooseX/SemiAffordanceAccessor/Role/Attribute.pm
+++ libmoosex-semiaffordanceaccessor-perl/lib/MooseX/SemiAffordanceAccessor/Role/Attribute.pm
@@ -44,7 +44,7 @@
 
 =head1 NAME
 
-MooseX::SemiAffordanceAccessor::Role::Attribute
+MooseX::SemiAffordanceAccessor::Role::Attribute - Role tweaking reader and writer parameters
 
 =head1 VERSION
 
