Description: Add whatis entry to documentation
Origin: vendor
Forwarded: no
Author: Ansgar Burchardt <ansgar@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-04-21

--- libmoosex-semiaffordanceaccessor-perl.orig/lib/MooseX/SemiAffordanceAccessor/Role/Attribute.pm
+++ libmoosex-semiaffordanceaccessor-perl/lib/MooseX/SemiAffordanceAccessor/Role/Attribute.pm
@@ -44,7 +44,7 @@
 
 =head1 NAME
 
-MooseX::SemiAffordanceAccessor::Role::Attribute
+MooseX::SemiAffordanceAccessor::Role::Attribute - Role tweaking reader and writer parameters
 
 =head1 VERSION
 
