Description: missing mandatory name from model Group
Origin: commit:7676
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=51109
Forwarded: yes
Author: yvesago-guest <agostini@univ-metz.fr>
Last-Update: 2009-11-26

--- a/t/TestApp-Plugin-REST/t/03-format.t
+++ b/t/TestApp-Plugin-REST/t/03-format.t
@@ -152,7 +152,7 @@
 $mech->post( $URL . '/=/model/User', { name => "moose", email => 'moose@example.com' } );
 is($mech->status, 200, "create via POST to model worked");
 
-$mech->post( $URL . '/=/model/Group', { } );
+$mech->post( $URL . '/=/model/Group', { name => "moose" } );
 is($mech->status, 403, "create via POST to model with disallowed create action failed with 403");
 
 # on GET    '/=/search/*/**' => \&search_items;
