Description: fix FTBFS problem
Author: Apocalypse <perl@0ne.us>
Reviewed-by: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: yes
Bug-CPAN: http://rt.cpan.org/Public/Bug/Display.html?id=53609
--- a/t/05_no_breakpoints_ok.t
+++ b/t/05_no_breakpoints_ok.t
@@ -5,7 +5,7 @@
 BEGIN {
     use Test::Tester 0.09;
     use Test::More;
-    our $tests = 46;
+    our $tests = 54;
     eval "use Test::NoWarnings";
     $tests++ unless( $@ );
     plan tests => $tests;
--- a/t/06_all_files_no_breakpoints_ok.t
+++ b/t/06_all_files_no_breakpoints_ok.t
@@ -5,7 +5,7 @@
 BEGIN {
     use Test::Tester 0.09;
     use Test::More;
-    our $tests = 27;
+    our $tests = 37;
     eval "use Test::NoWarnings";
     $tests++ unless( $@ );
     plan tests => $tests;
@@ -21,46 +21,56 @@
     [
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./01_use.t',
         },
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./02_pod.t',
         },
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./04_all_perl_files.t',
         },
         {
             ok   => 0,
+            depth => 2,
             name => 'no breakpoint test of ./05_no_breakpoints_ok.t',
             diag => 'breakpoint found in ./05_no_breakpoints_ok.t: $DB::signal =1' . "\n",
         },
         {
             ok   => 0,
+            depth => 2,
             name => 'no breakpoint test of ./06_all_files_no_breakpoints_ok.t',
             diag => 'breakpoint found in ./06_all_files_no_breakpoints_ok.t: $DB::signal =1' . "\n",
         },
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./07_deprecated_warnings.t',
         },
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./08_deprecated.t',
         },
         {
             ok   => 0,
+            depth => 2,
             name => 'no breakpoint test of ./baz/foo.t',
             diag => 'breakpoint found in ./baz/foo.t: $DB::signal = 1' . "\n",
         },
         {
             ok   => 0,
+            depth => 2,
             name => 'no breakpoint test of ./baz/gzonk/foo.pl',
             diag => 'breakpoint found in ./baz/gzonk/foo.pl: $DB::single = 2' . "\n",
         },
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./baz/quux/Foo.pm',
         },
     ],
--- a/t/08_deprecated.t
+++ b/t/08_deprecated.t
@@ -11,7 +11,7 @@
 BEGIN {
     use Test::Tester 0.09;
     use Test::More;
-    our $tests = 17;
+    our $tests = 21;
     eval "use Test::NoWarnings";
     $tests++ unless( $@ );
     plan tests => $tests;
@@ -39,14 +39,17 @@
     [
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./01_use.t',
         },
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./02_pod.t',
         },
         {
             ok   => 1,
+            depth => 2,
             name => 'no breakpoint test of ./04_all_perl_files.t',
         },
     ],
