Description: spelling fix
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-05-22

--- a/lib/DBD/CSV.pm
+++ b/lib/DBD/CSV.pm
@@ -446,7 +446,7 @@
 
     $dbh = DBI->connect ("dbi:CSV:f_dir=");
 
-The prefered way of passing the arguments is by driver attributes:
+The preferred way of passing the arguments is by driver attributes:
 
     # specify most possible flags via driver flags
     $dbh = DBI->connect ("dbi:CSV:", undef, undef, {
