Author: gregor herrmann <gregoa@debian.org>
Description: the leading ellipsis breaks POD and manpage

--- a/lib/Net/LDAP/FAQ.pod
+++ b/lib/Net/LDAP/FAQ.pod
@@ -1223,9 +1223,9 @@
 
   print ldap_error_name($result->code);
 
-=head2 Ho do I reset a user's password ...
+=head2 Ho do I reset a user's password -
 
-=head3 Z<>... in most LDAP servers ?
+=head3 - in most LDAP servers ?
 
 Most LDAP servers use the standard userPassword attribute as the
 attribute to set when you want to change a user's password.
@@ -1301,7 +1301,7 @@
 
   $ldap->unbind();
 
-=head3 Z<>... in MS Active Directory ?
+=head3 - in MS Active Directory ?
 
 With Active Directory a user's password is stored in the unicodePwd
 attribute and changed using the regular modify operation.
