Description: Use Digest::SHA instead deprecated Digest::SHA1
Origin: vendor
Forwarded: https://rt.cpan.org/Public/Bug/Display.html?id=71342
Author: Julián Moreno Patiño <darkjunix@gmail.com>
Last-Update: 2011-09-29
--- a/lib/Crypt/DSA.pm
+++ b/lib/Crypt/DSA.pm
@@ -2,7 +2,7 @@
 
 use 5.006;
 use strict;
-use Digest::SHA1 qw( sha1 );
+use Digest::SHA qw( sha1 );
 use Carp qw( croak );
 use Crypt::DSA::KeyChain;
 use Crypt::DSA::Key;
@@ -225,7 +225,7 @@
 
 is a shorter way of writing
 
-    use Digest::SHA1 qw( sha1 );
+    use Digest::SHA qw( sha1 );
     my $sig = $dsa->sign(Digest => sha1( $message ), ... );
 
 =back
--- a/lib/Crypt/DSA/KeyChain.pm
+++ b/lib/Crypt/DSA/KeyChain.pm
@@ -2,7 +2,7 @@
 
 use strict;
 use Math::BigInt 1.78 try => 'GMP, Pari';
-use Digest::SHA1 qw( sha1 );
+use Digest::SHA qw( sha1 );
 use Carp qw( croak );
 use IPC::Open3;
 use File::Spec;
