Description: fix a spelling mistake
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-04-16

--- a/Coro.pm
+++ b/Coro.pm
@@ -1233,7 +1233,7 @@
 by the forks module, which gives you the (i-) threads API, just much
 faster).
 
-Sharing data is in the i-threads model is done by transfering data
+Sharing data is in the i-threads model is done by transferring data
 structures between threads using copying semantics, which is very slow -
 shared data simply does not exist. Benchmarks using i-threads which are
 communication-intensive show extremely bad behaviour with i-threads (in
--- a/README
+++ b/README
@@ -1008,7 +1008,7 @@
     evidenced by the forks module, which gives you the (i-) threads API,
     just much faster).
 
-    Sharing data is in the i-threads model is done by transfering data
+    Sharing data is in the i-threads model is done by transferring data
     structures between threads using copying semantics, which is very slow -
     shared data simply does not exist. Benchmarks using i-threads which are
     communication-intensive show extremely bad behaviour with i-threads (in
