Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: Spelling
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=69672
Last-Update: 2011-07-21
Reviewed-by: Gregor Hermann <gregoa@debian.org>
--- a/lib/CGI/Application/Plugin/DevPopup.pm
+++ b/lib/CGI/Application/Plugin/DevPopup.pm
@@ -264,7 +264,7 @@
 time. This means you should place it in a BEGIN{} block, or use SetEnv or
 PerlSetEnv (remember to set those before any PerlRequire or PerlModule lines).
 
-Absense of the environment variable turns this module into a no-op: while the
+Absence of the environment variable turns this module into a no-op: while the
 plugin and its plugins are still loaded, they won't modify your output.
 
 =head1 Available Plugins
