Author: Niko Tyni <ntyni@iki.fi>
Description: Don't link unnecessarily with -lnsl.

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -5,7 +5,7 @@
 WriteMakefile(
     'NAME'	=> 'Net::Patricia',
     'VERSION_FROM' => 'Patricia.pm', # finds $VERSION
-    'LIBS'	=> ['-lnsl'],   # e.g., '-lm' 
+    'LIBS'	=> [''],   # e.g., '-lm'
     'DEFINE'	=> '',     # e.g., '-DHAVE_SOMETHING' 
     'INC'	=> '',     # e.g., '-I/usr/include/other' 
     'MYEXTLIB'	=> 'libpatricia/libpatricia$(LIB_EXT)',     # e.g., '-I/usr/include/other' 
