Description: Mention how to configure module on Debian
Origin: vendor
Forwarded: not-needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-05-22

--- a/SessionX.pm
+++ b/SessionX.pm
@@ -572,8 +572,9 @@
 
 =item config
 
-Use predefiend config from Apache::SessionX::Config, which is defined by
-Makefile.PL
+Use predefiend config from Apache::SessionX::Config, which is configured
+at package installation time.  On Debian, these may be altered by running
+C<dpkg-reconfigure libapache-sessionx-perl>.
 
 
 =item object_store
