Description: fix POD whatis entry
Origin: vendor
Forwarded: no
Author: Krzysztof Krzyzaniak <eloy@kofeina.net>
Reviewed-by: Jonathan Yu <jawnsy@cpan.org>
Last-Update: 2011-07-20

--- a/lib/POE/Component/Client/HTTP/RequestFactory.pm
+++ b/lib/POE/Component/Client/HTTP/RequestFactory.pm
@@ -23,6 +23,14 @@
 use constant FCT_TIMEOUT         => 9;
 use constant DEBUG               => 0;
 use constant DEFAULT_BLOCK_SIZE  => 4096;
+ 
+=head1 NAME
+
+POE::Component::Client::HTTP::RequestFactory - an HTTP request factory object
+
+=head1 SYNOPSIS
+
+  # Used internally by POE::Component::Client::HTTP
 
 =head1 CONSTRUCTOR
 
