From: Ansgar Burchardt <ansgar@43-1.org>
Subject: Correct spelling error
Forwarded: http://git.shadowcat.co.uk/gitweb/gitweb.cgi?p=gitmo/MooseX-Types-Structured.git;a=commitdiff;h=67eec8f705f693670aba8d2810e30fcf59191f44

--- libmoosex-types-structured-perl.orig/lib/MooseX/Meta/TypeConstraint/Structured.pm
+++ libmoosex-types-structured-perl/lib/MooseX/Meta/TypeConstraint/Structured.pm
@@ -267,7 +267,7 @@
 
 =head2 type_constraints_equals
 
-Checks to see if the internal type contraints are equal.
+Checks to see if the internal type constraints are equal.
 
 =cut
 
--- libmoosex-types-structured-perl.orig/lib/MooseX/Types/Structured.pm
+++ libmoosex-types-structured-perl/lib/MooseX/Types/Structured.pm
@@ -128,7 +128,7 @@
 =head2 Comparing Parameterized types to Structured types
 
 Parameterized constraints are built into core Moose and you are probably already
-familar with the type constraints 'HashRef' and 'ArrayRef'.  Structured types
+familiar with the type constraints 'HashRef' and 'ArrayRef'.  Structured types
 have similar functionality, so their syntax is likewise similar. For example,
 you could define a parameterized constraint like:
 
