Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: Spelling mistake
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=69643
Last-Update: 2010-10-23
--- a/lib/Apache2/Reload.pm
+++ b/lib/Apache2/Reload.pm
@@ -360,7 +360,7 @@
 
 =head2 Unregistering a module
 
-In some cases, it might be necessary to explicitely stop reloading
+In some cases, it might be necessary to explicitly stop reloading
 a module.
 
   Apache2::Reload->unregister_module('Some::Module');
