Description: Fix a spelling error of 'endianness' in hparser.c
Origin: vendor
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=50811
Forwarded: yes
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>
--- a/hparser.c
+++ b/hparser.c
@@ -1847,7 +1847,7 @@
 		warn("Parsing of undecoded UTF-8 will give garbage when decoding entities");
 	    }
 	    if (utf8 && len >= 2 && strnEQ(beg, "\xFF\xFE", 2)) {
-		warn("Parsing string decoded with wrong endianess");
+		warn("Parsing string decoded with wrong endianness");
 	    }
 #endif
 	    if (!utf8 && len >= 4 &&
