From: Ansgar Burchardt <ansgar@43-1.org>
Date: Tue, 11 May 2010 23:17:26 +0900
Origin: vendor
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=57375
Subject: Fix spelling errors

--- libperl-critic-perl.orig/TODO.pod
+++ libperl-critic-perl/TODO.pod
@@ -119,7 +119,7 @@
 
 - Data section count
 
-Proposed implementation commited 15-Mar-2007 by wyant, about revision 3240.
+Proposed implementation committed 15-Mar-2007 by wyant, about revision 3240.
 
 
 =item * Detect 5.10 source and enable stuff for that.
--- libperl-critic-perl.orig/lib/Perl/Critic/Policy/Documentation/RequirePodAtEnd.pm
+++ libperl-critic-perl/lib/Perl/Critic/Policy/Documentation/RequirePodAtEnd.pm
@@ -103,7 +103,7 @@
 
     frobulate()
     Accepts:  A list of things to frobulate
-    Returns:  True if succesful
+    Returns:  True if successful
 
     =end comments
 
