Author: gregor herrmann <gregoa@debian.org>
Description: spelling fix
Origin: vendor
Forwarded: no

--- a/README
+++ b/README
@@ -85,7 +85,7 @@
     tarball.
 
     The "-i" option lets you pass the name of a single file and it will
-    treat it as an installer for that single file. Further, if the extention
+    treat it as an installer for that single file. Further, if the extension
     of the tarball is .tar.gz, the -i option is implied.
 
     For example, the following are equivalent.
--- a/lib/pip.pm
+++ b/lib/pip.pm
@@ -300,7 +300,7 @@
 single tarball.
 
 The C<-i> option lets you pass the name of a single file and it will treat
-it as an installer for that single file. Further, if the extention of the
+it as an installer for that single file. Further, if the extension of the
 tarball is .tar.gz, the B<-i> option is implied.
 
 For example, the following are equivalent.
