Description: add missing whatis entries
Author: Ansgar Burchardt <ansgar@debian.org>
Origin: vendor
Forwarded: no
--- a/lib/Parse/CPAN/Packages/Distribution.pm
+++ b/lib/Parse/CPAN/Packages/Distribution.pm
@@ -54,7 +54,7 @@
 
 =head1 NAME
 
-Parse::CPAN::Packages::Distribution
+Parse::CPAN::Packages::Distribution - represents a CPAN distribution
 
 =head1 DESCRIPTION
 
--- a/lib/Parse/CPAN/Packages/Package.pm
+++ b/lib/Parse/CPAN/Packages/Package.pm
@@ -51,7 +51,7 @@
 
 =head1 NAME
 
-Parse::CPAN::Packages::Package
+Parse::CPAN::Packages::Package - represents a CPAN package
 
 =head1 DESCRIPTION
 
