Description: spelling fix
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Reviewed-by: Jonathan Yu <jawnsy@cpan.org>
Last-Update: 2011-03-19
--- a/lib/Test/YAML/Meta.pm
+++ b/lib/Test/YAML/Meta.pm
@@ -40,7 +40,7 @@
 However, it would help greatly if you are able to pinpoint problems or even
 supply a patch.
 
-Fixes are dependant upon their severity and my availablity. Should a fix not
+Fixes are dependent upon their severity and my availability. Should a fix not
 be forthcoming, please feel free to (politely) remind me.
 
 =head1 SEE ALSO
