--- a/Syslog.pm
+++ b/Syslog.pm
@@ -833,9 +833,14 @@ Version 0.26
 
 =head1 SYNOPSIS
 
-    use Sys::Syslog;                          # all except setlogsock(), or:
-    use Sys::Syslog qw(:DEFAULT setlogsock);  # default set, plus setlogsock()
-    use Sys::Syslog qw(:standard :macros);    # standard functions, plus macros
+    use Sys::Syslog;
+            # all except setlogsock(), or:
+
+    use Sys::Syslog qw(:DEFAULT setlogsock);
+            # default set, plus setlogsock()
+
+    use Sys::Syslog qw(:standard :macros);
+            # standard functions, plus macros
 
     openlog $ident, $logopt, $facility;       # don't forget this
     syslog $priority, $format, @args;
@@ -971,11 +976,13 @@ C<$format> that ends in a C<":">.
 
 B<Examples>
 
-    syslog("info", $message);           # informational level
-    syslog(LOG_INFO, $message);         # informational level
-
-    syslog("info|local0", $message);        # information level, Local0 facility
-    syslog(LOG_INFO|LOG_LOCAL0, $message);  # information level, Local0 facility
+    # informational level
+    syslog("info", $message);
+    syslog(LOG_INFO, $message);
+
+    # information level, Local0 facility
+    syslog("info|local0", $message);
+    syslog(LOG_INFO|LOG_LOCAL0, $message);
 
 =over 4
 
@@ -1012,7 +1019,9 @@ Log everything except informational mess
 
 Log critical messages, errors and warnings: 
 
-    setlogmask( LOG_MASK(LOG_CRIT) | LOG_MASK(LOG_ERR) | LOG_MASK(LOG_WARNING) );
+    setlogmask( LOG_MASK(LOG_CRIT)
+              | LOG_MASK(LOG_ERR)
+              | LOG_MASK(LOG_WARNING) );
 
 Log all messages up to debug: 
 
