Description: fix paths to FastCGI scripts
Author: AGOSTINI Yves <agostini@univ-metz.fr>
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
Origin: vendor
Forwarded: not-needed
Last-Update: 2010-11-21
--- a/lib/CGI/Fast.pm
+++ b/lib/CGI/Fast.pm
@@ -152,7 +152,7 @@
 FastCGI scripts must end in the extension .fcgi.  For each script you
 install, you must add something like the following to srm.conf:
 
-    FastCgiServer /usr/etc/httpd/fcgi-bin/file_upload.fcgi -processes 2
+    FastCgiServer /usr/lib/cgi-bin/file_upload.fcgi -processes 2
 
 This instructs Apache to launch two copies of file_upload.fcgi at
 startup time.
@@ -170,7 +170,7 @@
 webserver to connect to an external FastCGI server, you would add the following
 to your srm.conf:
 
-    FastCgiExternalServer /usr/etc/httpd/fcgi-bin/file_upload.fcgi -host sputnik:8888
+    FastCgiExternalServer /usr/lib/cgi-bin/file_upload.fcgi -host sputnik:8888
 
 Two environment variables affect how the C<CGI::Fast> object is created,
 allowing C<CGI::Fast> to be used as an external FastCGI server.  (See C<FCGI>
@@ -191,7 +191,7 @@
 
 For example:
 
-    #!/usr/local/bin/perl    # must be a FastCGI version of perl!
+    #!/usr/bin/perl
     use CGI::Fast;
     &do_some_initialization();
     $ENV{FCGI_SOCKET_PATH} = "sputnik:8888";
