Description: Fix a couple of spelling mistakes.
Forwarded: no
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2011-02-24

--- a/README
+++ b/README
@@ -139,7 +139,7 @@
 
       public name => our %name;
 
-    However, this makes private object data accessable outside the class and
+    However, this makes private object data accessible outside the class and
     incurs a slight performance penalty when accessing the property hash
     directly; it is not recommended to do this unless you really need it for
     some specialized reason.
@@ -255,7 +255,7 @@
       Class::InsideOut::options( \%new_options );
       %current_options = Class::InsideOut::options();
 
-    The "options" function sets default options for use with all subsquent
+    The "options" function sets default options for use with all subsequent
     property definitions for the calling package. If called without
     arguments, this function will return the options currently in effect.
     When called with a hash reference of options, these will be joined with
--- a/lib/Class/InsideOut.pm
+++ b/lib/Class/InsideOut.pm
@@ -688,7 +688,7 @@
 
  public name => our %name;
 
-However, this makes private object data accessable outside the class and incurs
+However, this makes private object data accessible outside the class and incurs
 a slight performance penalty when accessing the property hash directly; it is
 not recommended to do this unless you really need it for some specialized
 reason.
@@ -807,7 +807,7 @@
  Class::InsideOut::options( \%new_options );
  %current_options = Class::InsideOut::options();
 
-The {options} function sets default options for use with all subsquent property
+The {options} function sets default options for use with all subsequent property
 definitions for the calling package.  If called without arguments, this
 function will return the options currently in effect.  When called with a hash
 reference of options, these will be joined with the existing defaults,
--- a/lib/Class/InsideOut.pod
+++ b/lib/Class/InsideOut.pod
@@ -185,7 +185,7 @@
 
   public name => our %name;
 
-However, this makes private object data accessable outside the class and incurs
+However, this makes private object data accessible outside the class and incurs
 a slight performance penalty when accessing the property hash directly; it is
 not recommended to do this unless you really need it for some specialized
 reason.
@@ -304,7 +304,7 @@
   Class::InsideOut::options( \%new_options );
   %current_options = Class::InsideOut::options();
 
-The C<<< options >>> function sets default options for use with all subsquent property
+The C<<< options >>> function sets default options for use with all subsequent property
 definitions for the calling package.  If called without arguments, this
 function will return the options currently in effect.  When called with a hash
 reference of options, these will be joined with the existing defaults,
