make the man pages

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -50,8 +50,6 @@ my %makehash=( 
         ZIPFLAGS    => " -v -o -r -9 ",
         COMPRESS    => "gzip -9 -f ",
     }, 
-    MAN1PODS        => {}, 
-    MAN3PODS        => {}, 
     PREREQ_PM       => { 'Compress::Zlib' => 1.0, }, 
     VERSION_FROM    => $versionfile,
     BINARY_LOCATION => q|PDF-API2-|.$PDF::API2::Version::CVersion{vPerl}.q|.ppm.tar.gz|,
