add whatis entries to all the man pages

--- a/lib/PDF/API2/Annotation.pm
+++ b/lib/PDF/API2/Annotation.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Annotation;
 
+=head1 NAME
+
+PDF::API2::Annotation - annotation management
+
+=cut
+
 BEGIN 
 {
 
--- a/lib/PDF/API2/Resource/XObject/Image/TIFF.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/TIFF.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Resource::XObject::Image::TIFF;
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Image::TIFF - tiff support for PDF::API2
+
+=cut
+
 BEGIN {
 
     use PDF::API2::Util;
--- a/lib/PDF/API2/Resource/XObject/Image.pm
+++ b/lib/PDF/API2/Resource/XObject/Image.pm
@@ -50,6 +50,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Image - Image support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Image->new $pdf, $name
 
 Returns a image-resource object. base class for all types of bitmap-images.
--- a/lib/PDF/API2/Resource/XObject/Image/GD.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/GD.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Image::GD - GD::Image support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Image::GD->new $pdf, $obj [, $name]
 
 Returns a image object from a GD::Image.
--- a/lib/PDF/API2/Resource/XObject/Image/GIF.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/GIF.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Image::GIF - GIF image support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Image::GIF->new $pdf, $file [, $name]
 
 Returns a gif-image object.
--- a/lib/PDF/API2/Resource/XObject/Image/JPEG.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/JPEG.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Image::JPEG - JPEG support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Image::JPEG->new $pdf, $file [, $name]
 
 Returns a jpeg-image object.
--- a/lib/PDF/API2/Resource/XObject/Image/PNG.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/PNG.pm
@@ -47,6 +47,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Image::PNG - PNG support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Image::PNG->new $pdf, $file [, $name]
 
 Returns a png-image object.
--- a/lib/PDF/API2/Resource/XObject/Image/PNM.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/PNM.pm
@@ -46,6 +46,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Image::PNM - PNM image support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Image::PNM->new $pdf, $file [, $name]
 
 Returns a pnm-image object.
@@ -646,4 +650,4 @@ pi1toppm(1), picttoppm(1), pjtoppm(1), q
 1), ppmdither(1), ppmforge(1), ppmhist(1), ppmmake(1), ppmpat(1), ppmquant(1), ppmquantall(
 1), ppmrelief(1), pnm(5), pgm(5), pbm(5)
 AUTHOR
-Copyright (C) 1989, 1991 by Jef Poskanzer.
\ No newline at end of file
+Copyright (C) 1989, 1991 by Jef Poskanzer.
--- a/lib/PDF/API2/Resource/XObject/Form.pm
+++ b/lib/PDF/API2/Resource/XObject/Form.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Form - form support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Form->new $pdf
 
 Returns a form-resource object. base class for all types of form-xobjects.
--- a/lib/PDF/API2/Resource/XObject/Form/BarCode.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/BarCode.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Form::BarCode - generic barcode support for PDF::API2 forms
+
 =item $res = PDF::API2::Resource::XObject::Form::BarCode->new $pdf, %opts
 
 Returns a barcode-form object.
--- a/lib/PDF/API2/Resource/XObject/Form/BarCode/code128.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/BarCode/code128.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Form::BarCode::code128 - code128 barcode support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Form::BarCode::code128->new $pdf, %opts
 
 Returns a code128 object. Use '-ean' to encode using EAN128 mode.
--- a/lib/PDF/API2/Resource/XObject/Form/Hybrid.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/Hybrid.pm
@@ -54,6 +54,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject::Form::Hybrid - hybrid form support for PDF::API2
+
 =item $res = PDF::API2::Resource::XObject::Form::Hybrid->new $pdf
 
 Returns a hybrid-form object.
--- a/lib/PDF/API2/Resource/Font/Postscript.pm
+++ b/lib/PDF/API2/Resource/Font/Postscript.pm
@@ -53,6 +53,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::Font::Postscript - PostScript font support for PDF::API2
+
 =item $font = PDF::API2::Resource::Font::Postscript->new @parameters
 
 Returns a adobe type1 font object (called from $pdf->psfont).
--- a/lib/PDF/API2/Resource/Shading.pm
+++ b/lib/PDF/API2/Resource/Shading.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::Shading - generic shading support for PDF::API2
+
 =item $cs = PDF::API2::Resource::Shading->new $pdf, $key, %parameters
 
 Returns a new shading object. base class for all shadings.
--- a/lib/PDF/API2/Resource/XObject.pm
+++ b/lib/PDF/API2/Resource/XObject.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::XObject - XObject support for PDF::API2
+
 =head1 $res = PDF::API2::Resource::XObject->new $pdf, $name
 
 Returns a xobject-resource object.
--- a/lib/PDF/API2/Resource/Pattern.pm
+++ b/lib/PDF/API2/Resource/Pattern.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::Pattern - patterns for PDF::API2
+
 =item $cs = PDF::API2::Resource::Pattern->new $pdf, $key, %parameters
 
 Returns a new pattern object. base class for all patterns.
--- a/lib/PDF/API2/Resource/ColorSpace.pm
+++ b/lib/PDF/API2/Resource/ColorSpace.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ColorSpace - base color space support class for PDF::API2
+
 =item $cs = PDF::API2::Resource::ColorSpace->new $pdf, $key, %parameters
 
 Returns a new colorspace object. base class for all colorspaces.
--- a/lib/PDF/API2/Resource/ColorSpace/DeviceN.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/DeviceN.pm
@@ -48,6 +48,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ColorSpace::DeviceN - color space support for PDF::API2
+
 =item $cs = PDF::API2::Resource::ColorSpace::DeviceN->new $pdf, $key, %parameters
 
 Returns a new colorspace object.
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed.pm
@@ -48,6 +48,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ColorSpace::Indexed - indexed color space support for PDF::API2
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed->new $pdf, $key, %parameters
 
 Returns a new colorspace object.
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed/ACTFile.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed/ACTFile.pm
@@ -50,6 +50,10 @@ BEGIN {
 
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ColorSpace::Indexed::ACTFile - adobe color table support for PDF::API2
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed::ACTFile->new $pdf, $actfile
 
 Returns a new colorspace object created from an adobe color table file (ACT/8BCT).
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed/Hue.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed/Hue.pm
@@ -50,6 +50,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ColorSpace::Indexed::Hue - hue-based collor support for PDF::API2
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed::Hue->new $pdf
 
 Returns a new colorspace object created based on various hues.
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed/WebColor.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed/WebColor.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ColorSpace::Indexed::WebColor - web color support for PDF::API2
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed::WebColor->new $pdf
 
 Returns a new colorspace object created from the web color palette.
--- a/lib/PDF/API2/Resource/ColorSpace/Separation.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Separation.pm
@@ -48,6 +48,11 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ColorSpace::Separation - separation colorspace support for PDF::API2
+
+
 =item $cs = PDF::API2::Resource::ColorSpace::Separation->new $pdf, $key, %parameters
 
 Returns a new colorspace object.
--- a/lib/PDF/API2/Resource/ExtGState.pm
+++ b/lib/PDF/API2/Resource/ExtGState.pm
@@ -49,6 +49,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::ExtGState - extgate support for PDF::API2
+
 =item $egs = PDF::API2::Resource::ExtGState->new @parameters
 
 Returns a new extgstate object (called from $pdf->extgstate).
--- a/lib/PDF/API2/Resource/Font.pm
+++ b/lib/PDF/API2/Resource/Font.pm
@@ -52,6 +52,10 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=head1 NAME
+
+PDF::API2::Resource::Font - Font support for PDF::API2
+
 =item $font->encodeByData $encoding
 
 Encodes the font in the specified byte-encoding.
--- a/lib/PDF/API2/Basic/TTF/Anchor.pm
+++ b/lib/PDF/API2/Basic/TTF/Anchor.pm
@@ -26,6 +26,12 @@
 #=======================================================================
 package PDF::API2::Basic::TTF::Anchor;
 
+=head1 NAME
+
+PDF::API2::Basic::TTF::Anchor - Anchor points for GPOS tables
+
+=cut
+
 =head1 TITLE
 
 PDF::API2::Basic::TTF::Anchor - Anchor points for GPOS tables
--- a/lib/PDF/API2/Basic/TTF/Coverage.pm
+++ b/lib/PDF/API2/Basic/TTF/Coverage.pm
@@ -26,7 +26,7 @@
 #=======================================================================
 package PDF::API2::Basic::TTF::Coverage;
 
-=head1 TITLE
+=head1 NAME
 
 PDF::API2::Basic::TTF::Coverage - Opentype coverage and class definition objects
 
--- a/lib/PDF/API2/Basic/TTF/Delta.pm
+++ b/lib/PDF/API2/Basic/TTF/Delta.pm
@@ -26,6 +26,12 @@
 #=======================================================================
 package PDF::API2::Basic::TTF::Delta;
 
+=head1 NAME
+
+PDF::API2::Basic::TTF::Delta - Opentype Device tables
+
+=cut
+
 =head1 TITLE
 
 SIL::TTF::Delta - Opentype Device tables
--- a/lib/PDF/API2/Basic/TTF/GPOS.pm
+++ b/lib/PDF/API2/Basic/TTF/GPOS.pm
@@ -26,6 +26,12 @@
 #=======================================================================
 package PDF::API2::Basic::TTF::GPOS;
 
+=head1 NAME
+
+PDF::API2::Basic::TTF::GPOS - support for Opentype GPOS tables in conjunction with TTOpen
+
+=cut
+
 =head1 TITLE
 
 PDF::API2::Basic::TTF::GPOS - Support for Opentype GPOS tables in conjunction with TTOpen
--- a/lib/PDF/API2/Content.pm
+++ b/lib/PDF/API2/Content.pm
@@ -33,6 +33,12 @@
 
 package PDF::API2::Content;
 
+=head1 NAME
+
+PDF::API2::Content - content management
+
+=cut
+
 BEGIN {
 
     use strict;
--- a/lib/PDF/API2/Content/Text.pm
+++ b/lib/PDF/API2/Content/Text.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Content::Text;
 
+=head1 NAME
+
+PDF::API2::Content::Text - content text management
+
+=cut
+
 BEGIN {
 
     use strict;
--- a/lib/PDF/API2/Outline.pm
+++ b/lib/PDF/API2/Outline.pm
@@ -33,6 +33,12 @@
 
 package PDF::API2::Outline;
 
+=head1 NAME
+
+PDF::API2::Outline - outline management
+
+=cut
+
 BEGIN {
     use strict;
     use vars qw( @ISA $VERSION);
--- a/lib/PDF/API2/Outlines.pm
+++ b/lib/PDF/API2/Outlines.pm
@@ -33,6 +33,12 @@
 
 package PDF::API2::Outlines;
 
+=head1 NAME
+
+PDF::API2::Outlines -- PDF::API2 outlines
+
+=cut
+
 BEGIN {
 
     use strict;
--- a/lib/PDF/API2/Page.pm
+++ b/lib/PDF/API2/Page.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Page;
 
+=head1 NAME
+
+PDF::API2::Page - page management
+
+=cut
+
 BEGIN {
 
     use strict;
--- a/lib/PDF/API2/Resource.pm
+++ b/lib/PDF/API2/Resource.pm
@@ -33,6 +33,12 @@
 
 package PDF::API2::Resource;
 
+=head1 NAME
+
+PDF::API2::Resource -- generic PDF::API2 resource class
+
+=cut
+
 BEGIN {
 
     use PDF::API2::Util;
--- a/lib/PDF/API2/Resource/CIDFont/TrueType.pm
+++ b/lib/PDF/API2/Resource/CIDFont/TrueType.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Resource::CIDFont::TrueType;
 
+=head1 NAME
+
+PDF::API2::Resource::CIDFont::TrueType - TrueType font support for PDF::API2
+
+=cut
+
 BEGIN {
 
     use utf8;
--- a/lib/PDF/API2/Resource/BaseFont.pm
+++ b/lib/PDF/API2/Resource/BaseFont.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Resource::BaseFont;
 
+=head1 NAME
+
+PDF::API2::Resource::BaseFont - base font class
+
+=cut
+
 BEGIN {
 
     use utf8;
--- a/lib/PDF/API2/Resource/CIDFont.pm
+++ b/lib/PDF/API2/Resource/CIDFont.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Resource::CIDFont;
 
+=head1 NAME
+
+PDF::API2::Resource::CIDFont - base class for CID fonts
+
+=cut
+
 BEGIN 
 {
 
--- a/lib/PDF/API2/Resource/CIDFont/CJKFont.pm
+++ b/lib/PDF/API2/Resource/CIDFont/CJKFont.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Resource::CIDFont::CJKFont;
 
+=head1 NAME
+
+PDF::API2::Resource::CIDFont::CJKFont - base class for CJK fonts
+
+=cut
+
 BEGIN {
 
     use utf8;
--- a/lib/PDF/API2/Basic/TTF/Kern/Subtable.pm
+++ b/lib/PDF/API2/Basic/TTF/Kern/Subtable.pm
@@ -28,7 +28,7 @@ package PDF::API2::Basic::TTF::Kern::Sub
 
 =head1 NAME
 
-PDF::API2::Basic::TTF::Kern::Subtable
+PDF::API2::Basic::TTF::Kern::Subtable -- Kerning container class
 
 =head1 METHODS
 
--- a/lib/PDF/API2/NamedDestination.pm
+++ b/lib/PDF/API2/NamedDestination.pm
@@ -45,7 +45,7 @@ package PDF::API2::NamedDestination;
 
 =head1 NAME
 
-PDF::API2::NamedDestination
+PDF::API2::NamedDestination -- PDF::API2 named destination support
 
 =head1 SYNOPSIS
 
--- a/lib/PDF/API2/Basic/TTF/Mort/Subtable.pm
+++ b/lib/PDF/API2/Basic/TTF/Mort/Subtable.pm
@@ -28,7 +28,7 @@ package PDF::API2::Basic::TTF::Mort::Sub
 
 =head1 NAME
 
-PDF::API2::Basic::TTF::Mort::Subtable
+PDF::API2::Basic::TTF::Mort::Subtable -- base Mort class
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Basic/TTF/AATutils.pm
+++ b/lib/PDF/API2/Basic/TTF/AATutils.pm
@@ -26,6 +26,12 @@
 #=======================================================================
 package PDF::API2::Basic::TTF::AATutils;
 
+=head1 NAME
+
+PDF::API2::Basec::TTF::AATutils - misc TTF utils
+
+=cut
+
 use strict;
 use vars qw(@ISA @EXPORT);
 require Exporter;
--- a/lib/PDF/API2/Resource/UniFont.pm
+++ b/lib/PDF/API2/Resource/UniFont.pm
@@ -32,6 +32,12 @@
 #=======================================================================
 package PDF::API2::Resource::UniFont;
 
+=head1 NAME
+
+PDF::API2::Resource::UniFont - support class for PDF::API2
+
+=cut
+
 BEGIN {
 
     use utf8;
--- a/lib/PDF/API2/Version.pm
+++ b/lib/PDF/API2/Version.pm
@@ -28,7 +28,7 @@ __END__
 
 =head1 NAME
 
-PDF::API2::Version
+PDF::API2::Version - Autogenerated version information for PDF::API2
 
 =head1 VERSION
 
