fix typos

--- a/lib/PDF/API2/Content.pm
+++ b/lib/PDF/API2/Content.pm
@@ -1718,7 +1718,7 @@ Applys text to the content and optionall
 
 Options
 
-=ovar 4
+=over 4
 
 =item -indent
 
--- a/lib/PDF/API2/Basic/TTF/Table.pm
+++ b/lib/PDF/API2/Basic/TTF/Table.pm
@@ -343,7 +343,7 @@ sub empty
 
 Releases ALL of the memory used by this table, and all of its component/child
 objects.  This method is called automatically by
-'C<PDF::API2::Basic::TTF::Font-E<GT>release>' (so you don't have to call it yourself).
+'C<PDF::API2::Basic::TTF::Font-E<gt>release>' (so you don't have to call it yourself).
 
 B<NOTE>, that it is important that this method get called at some point prior
 to the actual destruction of the object.  Internally, we track things in a
