fix mistakes in the pod

--- a/lib/PDF/API2/Resource/XObject.pm
+++ b/lib/PDF/API2/Resource/XObject.pm
@@ -75,6 +75,8 @@ sub new {
     return($self);
 }
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject->new_api $api, $name
 
 Returns a xobject resource object. This method is different from 'new' that
@@ -120,6 +122,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Basic/PDF/Dict.pm
+++ b/lib/PDF/API2/Basic/PDF/Dict.pm
@@ -54,6 +54,8 @@ PDF::API2::Basic::PDF::Dict - PDF Dictio
 There are various special instance variables which are used to look after,
 particularly, streams. Each begins with a space:
 
+=over 4
+
 =item stream
 
 Holds the stream contents for output
@@ -68,6 +70,8 @@ not the same as a PDF file stream. The d
 If both ' stream' and ' streamfile' are empty, this indicates where in the
 source PDF the stream starts.
 
+=back
+
 =head1 METHODS
 
 =cut
--- a/lib/PDF/API2/Resource/XObject/Image/PNM.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/PNM.pm
@@ -50,6 +50,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Image::PNM - PNM image support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Image::PNM->new $pdf, $file [, $name]
 
 Returns a pnm-image object.
@@ -235,6 +237,8 @@ sub read_pnm {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Image/PNG.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/PNG.pm
@@ -51,6 +51,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Image::PNG - PNG support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Image::PNG->new $pdf, $file [, $name]
 
 Returns a png-image object.
@@ -366,6 +368,8 @@ sub unprocess {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Image/TIFF.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/TIFF.pm
@@ -56,6 +56,8 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=over 4
+
 =item  $res = PDF::API2::Resource::XObject::Image::TIFF->new $pdf, $file [, $name]
 
 Returns a tiff-image object.
@@ -371,6 +373,8 @@ B<Useful Tags:>
     xRes, yRes (dpi; pixel/cm if resUnit==3)
     resUnit
 
+=back
+
 =cut
 
 sub tiffTag {
--- a/lib/PDF/API2/Resource/BaseFont.pm
+++ b/lib/PDF/API2/Resource/BaseFont.pm
@@ -57,6 +57,8 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=over 4
+
 =item $font = PDF::API2::Resource::BaseFont->new $pdf, $name
 
 Returns a font resource object.
--- a/lib/PDF/API2/Annotation.pm
+++ b/lib/PDF/API2/Annotation.pm
@@ -84,6 +84,8 @@ sub outobjdeep 
     $self->SUPER::outobjdeep(@opts);
 }
 
+=over 4
+
 =item $ant->link $page, %opts
 
 Defines the annotation as launch-page with page $page and
@@ -430,6 +432,8 @@ sub dest 
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Basic/PDF/Array.pm
+++ b/lib/PDF/API2/Basic/PDF/Array.pm
@@ -44,10 +44,14 @@ This object is not an array but an assoc
 elements. Thus, there are special instance variables for an array object, beginning
 with a space
 
+=over 4
+
 =item var
 
 Contains the actual array of elements
 
+=back
+
 =head1 METHODS
 
 =head2 PDF::Array->new($parent, @vals)
--- a/lib/PDF/API2/Resource/XObject/Image/JPEG.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/JPEG.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Image::JPEG - JPEG support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Image::JPEG->new $pdf, $file [, $name]
 
 Returns a jpeg-image object.
@@ -170,6 +172,8 @@ sub read_jpeg {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Image/GIF.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/GIF.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Image::GIF - GIF image support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Image::GIF->new $pdf, $file [, $name]
 
 Returns a gif-image object.
@@ -284,6 +286,8 @@ sub new_api {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Image/GD.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/GD.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Image::GD - GD::Image support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Image::GD->new $pdf, $obj [, $name]
 
 Returns a image object from a GD::Image.
@@ -138,6 +140,8 @@ sub read_gd {
 
 __END__
 
+=back 
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Form.pm
+++ b/lib/PDF/API2/Resource/XObject/Form.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Form - form support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Form->new $pdf
 
 Returns a form-resource object. base class for all types of form-xobjects.
@@ -160,6 +162,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Form/BarCode.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/BarCode.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Form::BarCode - generic barcode support for PDF::API2 forms
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Form::BarCode->new $pdf, %opts
 
 Returns a barcode-form object.
@@ -259,6 +261,8 @@ sub height {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Form/Hybrid.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/Hybrid.pm
@@ -58,6 +58,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Form::Hybrid - hybrid form support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Form::Hybrid->new $pdf
 
 Returns a hybrid-form object.
@@ -132,6 +134,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Image.pm
+++ b/lib/PDF/API2/Resource/XObject/Image.pm
@@ -54,6 +54,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Image - Image support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Image->new $pdf, $name
 
 Returns a image-resource object. base class for all types of bitmap-images.
@@ -190,6 +192,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/XObject/Form/BarCode/code128.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/BarCode/code128.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::XObject::Form::BarCode::code128 - code128 barcode support for PDF::API2
 
+=over 4
+
 =item $res = PDF::API2::Resource::XObject::Form::BarCode::code128->new $pdf, %opts
 
 Returns a code128 object. Use '-ean' to encode using EAN128 mode.
@@ -233,6 +235,8 @@ sub encode_ean128 {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/Font.pm
+++ b/lib/PDF/API2/Resource/Font.pm
@@ -56,6 +56,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::Font - Font support for PDF::API2
 
+=over 4
+
 =item $font->encodeByData $encoding
 
 Encodes the font in the specified byte-encoding.
@@ -265,6 +267,8 @@ sub remap {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/Pattern.pm
+++ b/lib/PDF/API2/Resource/Pattern.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::Pattern - patterns for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::Pattern->new $pdf, $key, %parameters
 
 Returns a new pattern object. base class for all patterns.
@@ -100,6 +102,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/Shading.pm
+++ b/lib/PDF/API2/Resource/Shading.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::Shading - generic shading support for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::Shading->new $pdf, $key, %parameters
 
 Returns a new shading object. base class for all shadings.
@@ -110,6 +112,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/UniFont.pm
+++ b/lib/PDF/API2/Resource/UniFont.pm
@@ -54,6 +54,8 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=over 4
+
 =item $font = PDF::API2::Resource::UniFont->new $pdf, @fontspecs, %options
 
 Returns a uni-font object.
@@ -325,3 +327,5 @@ sub text 
 1;
 
 __END__
+
+=back
--- a/lib/PDF/API2/Resource/Font/CoreFont.pm
+++ b/lib/PDF/API2/Resource/Font/CoreFont.pm
@@ -427,6 +427,8 @@ __END__
 
 =head1 SUPPORTED FONTS
 
+=over 4
+
 =item PDF::API::CoreFont supports the following 'Adobe Core Fonts':
 
   Courier
@@ -457,6 +459,8 @@ __END__
   Webdings
   Wingdings
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/Font/Postscript.pm
+++ b/lib/PDF/API2/Resource/Font/Postscript.pm
@@ -57,6 +57,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::Font::Postscript - PostScript font support for PDF::API2
 
+=over 4
+
 =item $font = PDF::API2::Resource::Font::Postscript->new @parameters
 
 Returns a adobe type1 font object (called from $pdf->psfont).
@@ -564,6 +566,8 @@ sub readXFM {
 
 __END__
 
+=back
+
 =head1 HISTORY
 
     $Log: Postscript.pm,v $
--- a/lib/PDF/API2/Resource/ColorSpace.pm
+++ b/lib/PDF/API2/Resource/ColorSpace.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ColorSpace - base color space support class for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::ColorSpace->new $pdf, $key, %parameters
 
 Returns a new colorspace object. base class for all colorspaces.
@@ -132,6 +134,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/ExtGState.pm
+++ b/lib/PDF/API2/Resource/ExtGState.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ExtGState - extgate support for PDF::API2
 
+=over 4
+
 =item $egs = PDF::API2::Resource::ExtGState->new @parameters
 
 Returns a new extgstate object (called from $pdf->extgstate).
@@ -407,6 +409,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/ColorSpace/DeviceN.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/DeviceN.pm
@@ -52,6 +52,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ColorSpace::DeviceN - color space support for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::ColorSpace::DeviceN->new $pdf, $key, %parameters
 
 Returns a new colorspace object.
@@ -150,6 +152,8 @@ sub param {
 
 __END__
 
+=back
+
 =head1 HISTORY
 
     $Log: DeviceN.pm,v $
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed.pm
@@ -52,6 +52,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ColorSpace::Indexed - indexed color space support for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed->new $pdf, $key, %parameters
 
 Returns a new colorspace object.
@@ -123,6 +125,8 @@ sub resolveNearestRGB {
 
 __END__
 
+=back
+
 =head1 HISTORY
 
     $Log: Indexed.pm,v $
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed/ACTFile.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed/ACTFile.pm
@@ -54,6 +54,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ColorSpace::Indexed::ACTFile - adobe color table support for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed::ACTFile->new $pdf, $actfile
 
 Returns a new colorspace object created from an adobe color table file (ACT/8BCT).
@@ -115,6 +117,8 @@ sub new_api {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed/Hue.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed/Hue.pm
@@ -54,6 +54,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ColorSpace::Indexed::Hue - hue-based collor support for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed::Hue->new $pdf
 
 Returns a new colorspace object created based on various hues.
@@ -116,6 +118,8 @@ sub new_api {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed/WebColor.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed/WebColor.pm
@@ -53,6 +53,8 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ColorSpace::Indexed::WebColor - web color support for PDF::API2
 
+=over 4
+
 =item $cs = PDF::API2::Resource::ColorSpace::Indexed::WebColor->new $pdf
 
 Returns a new colorspace object created from the web color palette.
@@ -126,6 +128,8 @@ sub new_api {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/ColorSpace/Separation.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Separation.pm
@@ -52,6 +52,7 @@ no warnings qw[ deprecated recursion uni
 
 PDF::API2::Resource::ColorSpace::Separation - separation colorspace support for PDF::API2
 
+=over 4
 
 =item $cs = PDF::API2::Resource::ColorSpace::Separation->new $pdf, $key, %parameters
 
@@ -196,6 +197,8 @@ sub param {
 
 __END__
 
+=back
+
 =head1 HISTORY
 
     $Log: Separation.pm,v $
--- a/lib/PDF/API2/Resource/CIDFont/CJKFont.pm
+++ b/lib/PDF/API2/Resource/CIDFont/CJKFont.pm
@@ -62,6 +62,8 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=over 4
+
 =item $font = PDF::API2::Resource::CIDFont::CJKFont->new $pdf, $cjkname, %options
 
 Returns a cjk-font object.
@@ -380,6 +382,8 @@ BEGIN {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/CIDFont/TrueType.pm
+++ b/lib/PDF/API2/Resource/CIDFont/TrueType.pm
@@ -61,6 +61,8 @@ BEGIN {
 }
 no warnings qw[ deprecated recursion uninitialized ];
 
+=over 4
+
 =item $font = PDF::API2::Resource::CIDFont::TrueType->new $pdf, $file, %options
 
 Returns a font object.
@@ -244,6 +246,8 @@ sub outobjdeep 
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Content.pm
+++ b/lib/PDF/API2/Content.pm
@@ -112,6 +112,8 @@ sub outobjdeep {
     $self->SUPER::outobjdeep(@_);
 }
 
+=over 4
+
 =item $co->add @content
 
 Adds @content to the object.
@@ -665,6 +667,8 @@ sub strokecolor 
     return(@{$self->{' strokecolor'}});
 }
 
+=back
+
 =head1 GRAPHICS METHODS
 
 =over 4
@@ -2199,6 +2203,8 @@ sub resource 
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Lite.pm
+++ b/lib/PDF/API2/Lite.pm
@@ -66,6 +66,8 @@ PDF::API2::Lite - lite pdf creation
 
 =head2 PDF::API2::Lite
 
+=over 4
+
 =item $pdf = PDF::API::Lite->new
 
 =cut
@@ -663,6 +665,8 @@ sub print {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/NamedDestination.pm
+++ b/lib/PDF/API2/NamedDestination.pm
@@ -324,6 +324,8 @@ sub dest 
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Outline.pm
+++ b/lib/PDF/API2/Outline.pm
@@ -121,6 +121,8 @@ sub fix_outline {
     $self->count;
 }
 
+=over 4
+
 =item $otl->title $text
 
 Set the title of the outline.
@@ -378,6 +380,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Page.pm
+++ b/lib/PDF/API2/Page.pm
@@ -84,6 +84,8 @@ sub new {
     $self;
 }
 
+=over 4
+
 =item $page = PDF::API2::Page->coerce $pdf, $pdfpage
 
 Returns a page object converted from $pdfpage (called from $pdf->openpage).
@@ -560,6 +562,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource.pm
+++ b/lib/PDF/API2/Resource.pm
@@ -78,6 +78,8 @@ sub new {
     return($self);
 }
 
+=over 4
+
 =item $res = PDF::API2::Resource->new_api $api, $name
 
 Returns a resource object. This method is different from 'new' that
@@ -123,6 +125,8 @@ sub outobjdeep {
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Resource/CIDFont.pm
+++ b/lib/PDF/API2/Resource/CIDFont.pm
@@ -59,6 +59,8 @@ BEGIN 
 
 no warnings qw[ deprecated recursion uninitialized ];
 
+=over 4
+
 =item $font = PDF::API2::Resource::CIDFont->new $pdf, $name
 
 Returns a cid-font object. base class form all CID based fonts.
@@ -405,6 +407,8 @@ sub outobjdeep 
 
 __END__
 
+=back
+
 =head1 AUTHOR
 
 alfred reibenschuh
--- a/lib/PDF/API2/Basic/TTF/OldMort.pm
+++ b/lib/PDF/API2/Basic/TTF/OldMort.pm
@@ -34,6 +34,8 @@ PDF::API2::Basic::TTF::OldMort - Glyph M
 
 =head1 INSTANCE VARIABLES
 
+=over 4
+
 =item version
 
 table version number (Fixed: currently 1.0)
@@ -137,6 +139,8 @@ Insertion table:
 
 =back
 
+=back
+
 =head1 METHODS
 
 =cut
--- a/lib/PDF/API2/Basic/TTF/Prop.pm
+++ b/lib/PDF/API2/Basic/TTF/Prop.pm
@@ -34,6 +34,8 @@ PDF::API2::Basic::TTF::Prop - Glyph Prop
 
 =head1 INSTANCE VARIABLES
 
+=over 4
+
 =item version
 
 =item default
@@ -44,6 +46,8 @@ Hash of property values keyed by glyph n
 
 =item lookupFormat
 
+=back
+
 =head1 METHODS
 
 =cut
--- a/lib/PDF/API2/Basic/TTF/XMLparse.pm
+++ b/lib/PDF/API2/Basic/TTF/XMLparse.pm
@@ -81,9 +81,9 @@ Nothing is stored in the parent. (This r
 
 Anything else is taken as the element content to be stored in the parent.
 
-=back 4
+=back
 
-=back 4
+=back
 
 In addition, the context hash passed to these messages contains the following
 keys:
@@ -124,7 +124,7 @@ whether we are dealing with an array or 
 Character processing is to insert all the characters into the text element of the
 context for available use later.
 
-=back 4
+=back
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Basic/TTF/Coverage.pm
+++ b/lib/PDF/API2/Basic/TTF/Coverage.pm
@@ -38,6 +38,8 @@ say that a coverage table is a class def
 for each glyph is the corresponding index in the coverage table. The resulting
 data structure is that a Coverage table has the following fields:
 
+=over 4
+
 =item cover
 
 A boolean to indicate whether this table is a coverage table (TRUE) or a
@@ -57,6 +59,8 @@ is written out.
 A count of the elements in a coverage table for use with add. Each subsequent
 addition is added with the current count and increments the count.
 
+=back
+
 =head1 METHODS
 
 =cut
--- a/lib/PDF/API2/Basic/TTF/Delta.pm
+++ b/lib/PDF/API2/Basic/TTF/Delta.pm
@@ -41,6 +41,8 @@ SIL::TTF::Delta - Opentype Device tables
 Each device table corresponds to a set of deltas for a particular point over
 a range of ppem values.
 
+=over 4
+
 =item first
 
 The first ppem value in the range
@@ -59,6 +61,8 @@ first and last inclusive.
 This is the fmt used (log2 of number bits per value) when the device table was
 read. It is recalculated on output.
 
+=back
+
 =head1 METHODS
 
 =cut
--- a/lib/PDF/API2/Basic/TTF/Fdsc.pm
+++ b/lib/PDF/API2/Basic/TTF/Fdsc.pm
@@ -34,12 +34,16 @@ PDF::API2::Basic::TTF::Fdsc - Font Descr
 
 =head1 INSTANCE VARIABLES
 
+=over 4
+
 =item version
 
 =item descriptors
 
 Hash keyed by descriptor tags
 
+=back
+
 =head1 METHODS
 
 =cut
--- a/lib/PDF/API2/Basic/TTF/Kern.pm
+++ b/lib/PDF/API2/Basic/TTF/Kern.pm
@@ -61,10 +61,10 @@ Number of subtables in the kerning table
 
 Array of subtables in the kerning table
 
-=over 4
-
 Each subtable has a number of instance variables.
 
+=over 4
+
 =item kern
 
 A two level hash array containing kerning values. The indexing is left
--- a/lib/PDF/API2/Basic/TTF/Bsln.pm
+++ b/lib/PDF/API2/Basic/TTF/Bsln.pm
@@ -34,6 +34,8 @@ PDF::API2::Basic::TTF::Bsln - Baseline t
 
 =head1 INSTANCE VARIABLES
 
+=over 4
+
 =item version
 
 =item xformat
@@ -50,6 +52,8 @@ PDF::API2::Basic::TTF::Bsln - Baseline t
 
 =item lookup
 
+=back
+
 =head1 METHODS
 
 =cut
--- a/lib/PDF/API2/Basic/PDF/Page.pm
+++ b/lib/PDF/API2/Basic/PDF/Page.pm
@@ -50,10 +50,14 @@ structurally quite different.
 
 A page has various working variables:
 
+=over 4
+
 =item curstrm
 
 The currently open stream
 
+=back
+
 =head1 METHODS
 
 =head2 PDF::API2::Basic::PDF::Page->new($pdf, $parent, $index)
