Author: gregor herrmann <gregoa@debian.org>
Description: make man happy about whatis part
Index: libmail-sendmail-perl/Sendmail.pm
===================================================================
--- libmail-sendmail-perl.orig/Sendmail.pm	2008-08-26 16:13:17.000000000 -0500
+++ libmail-sendmail-perl/Sendmail.pm	2008-08-26 16:14:24.000000000 -0500
@@ -5,7 +5,7 @@
 
 =head1 NAME
 
-Mail::Sendmail v. 0.79_16 - Simple platform independent mailer
+Mail::Sendmail - Simple platform independent mailer
 
 =cut
 
