Author: gregor herrmann <gregoa@debian.org>
Bugs-Debian: #537720
Description: fix POD, whitespace in L<> is significant

--- a/lib/XML/XPathEngine.pm
+++ b/lib/XML/XPathEngine.pm
@@ -1169,7 +1169,7 @@
 
 L<Tree::XPathEngine> for a similar module for non-XML trees.
 
-L<http://www.xmltwig.com/article/extending_xml_xpath/ > for background 
+L<http://www.xmltwig.com/article/extending_xml_xpath/> for background
 information. The last section of the article summarizes how to reuse XML::XPath.
 As XML::XPathEngine offers the same API it should help you
 
