Description: spelling fixes
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-01-05

--- a/lib/Crypt/DH/GMP.pm
+++ b/lib/Crypt/DH/GMP.pm
@@ -71,7 +71,7 @@
 C<Crypt::DH> works with a plain C<Math::BigInt>, but if you want to use
 it in production, you almost always need to install C<Math::BigInt::GMP>
 or C<Math::BigInt::Pari> because without them, the computation that is
-required by C<Crypt::DH> makes the module prettu much unusable.
+required by C<Crypt::DH> makes the module pretty much unusable.
 
 Because of this, C<Crypt::DH> might as well make C<Math::BigInt::GMP> a
 hard requirement.
@@ -91,7 +91,7 @@
 These problems quickly become apparent when you use modules such as 
 C<Net::OpenID::Consumer>, which requires to make a few calls to C<Crypt::DH>.
 
-C<Crypt::DH::GMP> attemps to alleviate these problems by providing a 
+C<Crypt::DH::GMP> attempts to alleviate these problems by providing a
 C<Crypt::DH>-compatible layer, which, instead of doing calculations via
 Math::BigInt, directly works with libgmp in C.
 
