Description: Fix spelling errors in manpage reported by lintian.
Origin: vendor
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=54870
Forwarded: yes
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>
--- a/lib/RRDTool/OO.pm
+++ b/lib/RRDTool/OO.pm
@@ -1287,7 +1287,7 @@
 to combine several primary data points into one archive point, specify
 values for 
 C<cpoints> (the number of points to combine) and C<cfunc> 
-(the consolidation function) explicitely:
+(the consolidation function) explicitly:
 
     $rrd->create(
          step        => 60,
@@ -1444,7 +1444,7 @@
     );
 
 This will assume a start time of 24 hours before now and an
-end time of now. Specify C<start> and C<end> explicitely to
+end time of now. Specify C<start> and C<end> explicitly to
 be clear:
 
     $rrd->graph(
@@ -1462,7 +1462,7 @@
 not specified. The values for data source and consolidation function
 default to the first values it finds in the RRD.
 If there are multiple datasources in the RRD or multiple archives
-with different values for C<cfunc>, just specify explicitely which
+with different values for C<cfunc>, just specify explicitly which
 one to draw:
 
     $rrd->graph(
@@ -2017,7 +2017,7 @@
         level    => $DEBUG
     }); 
 
-If you're interested particularily in I<rrdtool> commands issued
+If you're interested particularly in I<rrdtool> commands issued
 by C<RRDTool::OO> while you're operating it, just enable the
 category C<"rrdtool">:
 
